2023-12-12 16:36:54.080 WARNING (SyncWorker_0) [homeassistant.loader] We found a custom integration breaking_changes which has not been tested by Home Assistant. This component might cause stability problems, be sure to disable it if you experience issues with Home Assistant
2023-12-12 16:36:54.085 WARNING (SyncWorker_0) [homeassistant.loader] We found a custom integration plex_recently_added which has not been tested by Home Assistant. This component might cause stability problems, be sure to disable it if you experience issues with Home Assistant
2023-12-12 16:36:54.088 WARNING (SyncWorker_0) [homeassistant.loader] We found a custom integration manish which has not been tested by Home Assistant. This component might cause stability problems, be sure to disable it if you experience issues with Home Assistant
2023-12-12 16:36:54.090 WARNING (SyncWorker_0) [homeassistant.loader] We found a custom integration plex_assistant which has not been tested by Home Assistant. This component might cause stability problems, be sure to disable it if you experience issues with Home Assistant
2023-12-12 16:36:54.093 WARNING (SyncWorker_0) [homeassistant.loader] We found a custom integration anniversaries which has not been tested by Home Assistant. This component might cause stability problems, be sure to disable it if you experience issues with Home Assistant
2023-12-12 16:36:54.095 WARNING (SyncWorker_0) [homeassistant.loader] We found a custom integration ourgroceries which has not been tested by Home Assistant. This component might cause stability problems, be sure to disable it if you experience issues with Home Assistant
2023-12-12 16:36:54.097 WARNING (SyncWorker_0) [homeassistant.loader] We found a custom integration average which has not been tested by Home Assistant. This component might cause stability problems, be sure to disable it if you experience issues with Home Assistant
2023-12-12 16:36:54.099 WARNING (SyncWorker_0) [homeassistant.loader] We found a custom integration afvalinfo which has not been tested by Home Assistant. This component might cause stability problems, be sure to disable it if you experience issues with Home Assistant
2023-12-12 16:36:54.101 WARNING (SyncWorker_0) [homeassistant.loader] We found a custom integration car_wash which has not been tested by Home Assistant. This component might cause stability problems, be sure to disable it if you experience issues with Home Assistant
2023-12-12 16:36:54.104 WARNING (SyncWorker_0) [homeassistant.loader] We found a custom integration saver which has not been tested by Home Assistant. This component might cause stability problems, be sure to disable it if you experience issues with Home Assistant
2023-12-12 16:36:54.105 WARNING (SyncWorker_0) [homeassistant.loader] We found a custom integration buienalarm which has not been tested by Home Assistant. This component might cause stability problems, be sure to disable it if you experience issues with Home Assistant
2023-12-12 16:36:54.108 WARNING (SyncWorker_0) [homeassistant.loader] We found a custom integration hacs which has not been tested by Home Assistant. This component might cause stability problems, be sure to disable it if you experience issues with Home Assistant
2023-12-12 16:36:54.110 WARNING (SyncWorker_0) [homeassistant.loader] We found a custom integration favicon which has not been tested by Home Assistant. This component might cause stability problems, be sure to disable it if you experience issues with Home Assistant
2023-12-12 16:36:54.112 WARNING (SyncWorker_0) [homeassistant.loader] We found a custom integration temperature_feels_like which has not been tested by Home Assistant. This component might cause stability problems, be sure to disable it if you experience issues with Home Assistant
2023-12-12 16:36:54.114 WARNING (SyncWorker_0) [homeassistant.loader] We found a custom integration illuminance which has not been tested by Home Assistant. This component might cause stability problems, be sure to disable it if you experience issues with Home Assistant
2023-12-12 16:36:54.116 WARNING (SyncWorker_0) [homeassistant.loader] We found a custom integration hpprinter which has not been tested by Home Assistant. This component might cause stability problems, be sure to disable it if you experience issues with Home Assistant
2023-12-12 16:36:54.123 WARNING (SyncWorker_0) [homeassistant.loader] We found a custom integration fontawesome which has not been tested by Home Assistant. This component might cause stability problems, be sure to disable it if you experience issues with Home Assistant
2023-12-12 16:36:54.125 WARNING (SyncWorker_0) [homeassistant.loader] We found a custom integration pyscript which has not been tested by Home Assistant. This component might cause stability problems, be sure to disable it if you experience issues with Home Assistant
2023-12-12 16:36:54.127 WARNING (SyncWorker_0) [homeassistant.loader] We found a custom integration simpleicons which has not been tested by Home Assistant. This component might cause stability problems, be sure to disable it if you experience issues with Home Assistant
2023-12-12 16:36:54.130 WARNING (SyncWorker_0) [homeassistant.loader] We found a custom integration google_home which has not been tested by Home Assistant. This component might cause stability problems, be sure to disable it if you experience issues with Home Assistant
2023-12-12 16:36:54.132 WARNING (SyncWorker_0) [homeassistant.loader] We found a custom integration afvalbeheer which has not been tested by Home Assistant. This component might cause stability problems, be sure to disable it if you experience issues with Home Assistant
2023-12-12 16:36:54.134 WARNING (SyncWorker_0) [homeassistant.loader] We found a custom integration worlds_air_quality_index which has not been tested by Home Assistant. This component might cause stability problems, be sure to disable it if you experience issues with Home Assistant
2023-12-12 16:36:54.137 WARNING (SyncWorker_0) [homeassistant.loader] We found a custom integration knmi which has not been tested by Home Assistant. This component might cause stability problems, be sure to disable it if you experience issues with Home Assistant
2023-12-12 16:36:54.139 WARNING (SyncWorker_0) [homeassistant.loader] We found a custom integration google_photos which has not been tested by Home Assistant. This component might cause stability problems, be sure to disable it if you experience issues with Home Assistant
2023-12-12 16:36:54.141 WARNING (SyncWorker_0) [homeassistant.loader] We found a custom integration n26 which has not been tested by Home Assistant. This component might cause stability problems, be sure to disable it if you experience issues with Home Assistant
2023-12-12 16:36:54.141 ERROR (SyncWorker_0) [homeassistant.loader] The custom integration 'n26' does not have a version key in the manifest file and was blocked from loading. See https://developers.home-assistant.io/blog/2021/01/29/custom-integration-changes#versions for more details
2023-12-12 16:36:54.144 WARNING (SyncWorker_0) [homeassistant.loader] We found a custom integration whatspie which has not been tested by Home Assistant. This component might cause stability problems, be sure to disable it if you experience issues with Home Assistant
2023-12-12 16:36:54.146 WARNING (SyncWorker_0) [homeassistant.loader] We found a custom integration lg_hombot which has not been tested by Home Assistant. This component might cause stability problems, be sure to disable it if you experience issues with Home Assistant
2023-12-12 16:36:54.148 WARNING (SyncWorker_0) [homeassistant.loader] We found a custom integration pollenprognos which has not been tested by Home Assistant. This component might cause stability problems, be sure to disable it if you experience issues with Home Assistant
2023-12-12 16:36:54.150 WARNING (SyncWorker_0) [homeassistant.loader] We found a custom integration jokes which has not been tested by Home Assistant. This component might cause stability problems, be sure to disable it if you experience issues with Home Assistant
2023-12-12 16:36:54.152 WARNING (SyncWorker_0) [homeassistant.loader] We found a custom integration auto_backup which has not been tested by Home Assistant. This component might cause stability problems, be sure to disable it if you experience issues with Home Assistant
2023-12-12 16:36:54.154 WARNING (SyncWorker_0) [homeassistant.loader] We found a custom integration spotcast which has not been tested by Home Assistant. This component might cause stability problems, be sure to disable it if you experience issues with Home Assistant
2023-12-12 16:37:15.339 ERROR (MainThread) [homeassistant.helpers.template] Template variable error: 'None' has no attribute 'attributes' when rendering '{{ states.vacuum.hombot.attributes.battery_icon }}'
2023-12-12 16:37:22.021 ERROR (SyncWorker_2) [glocaltokens.client] [!] Could not get access token.
2023-12-12 16:37:27.096 ERROR (SyncWorker_2) [homeassistant.components.command_line.utils] Command failed (with return code 1): python3 -c "import requests; import json; import random; dataRequest = requests.get('https://gpsgadget.buienradar.nl/data/raintext?lat=51.83&lon=5.86&c='+str(random.randint(0,999999999999999)) ).text; dataRequest = dataRequest.replace('\r\n',' '); data = '{\"data\":\"'+dataRequest+'\"}';    pri(data);"
2023-12-12 16:37:27.730 WARNING (MainThread) [homeassistant.helpers.template] Template variable warning: 'None' has no attribute 'last_changed' when rendering 'last_changed: {{states.sensor.neerslag_buienalarm_regen_data.last_changed}}'
2023-12-12 16:37:27.731 WARNING (MainThread) [homeassistant.components.command_line] Empty reply found when expecting JSON data
2023-12-12 16:37:56.598 WARNING (SyncWorker_4) [py.warnings] /usr/local/lib/python3.11/site-packages/urllib3/connectionpool.py:1061: InsecureRequestWarning: Unverified HTTPS request is being made to host '192.168.69.103'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/1.26.x/advanced-usage.html#ssl-warnings
  warnings.warn(

2023-12-12 16:37:56.609 WARNING (SyncWorker_4) [py.warnings] /usr/local/lib/python3.11/site-packages/urllib3/connectionpool.py:1061: InsecureRequestWarning: Unverified HTTPS request is being made to host '192.168.69.103'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/1.26.x/advanced-usage.html#ssl-warnings
  warnings.warn(

2023-12-12 16:37:57.571 WARNING (SyncWorker_1) [py.warnings] /usr/local/lib/python3.11/site-packages/urllib3/connectionpool.py:1061: InsecureRequestWarning: Unverified HTTPS request is being made to host '192.168.69.103'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/1.26.x/advanced-usage.html#ssl-warnings
  warnings.warn(

2023-12-12 16:37:57.581 WARNING (SyncWorker_1) [py.warnings] /usr/local/lib/python3.11/site-packages/urllib3/connectionpool.py:1061: InsecureRequestWarning: Unverified HTTPS request is being made to host '192.168.69.103'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/1.26.x/advanced-usage.html#ssl-warnings
  warnings.warn(

2023-12-12 16:37:57.667 WARNING (SyncWorker_0) [py.warnings] /usr/local/lib/python3.11/site-packages/urllib3/connectionpool.py:1061: InsecureRequestWarning: Unverified HTTPS request is being made to host '192.168.69.103'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/1.26.x/advanced-usage.html#ssl-warnings
  warnings.warn(

2023-12-12 16:37:57.676 WARNING (SyncWorker_5) [py.warnings] /usr/local/lib/python3.11/site-packages/urllib3/connectionpool.py:1061: InsecureRequestWarning: Unverified HTTPS request is being made to host '192.168.69.103'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/1.26.x/advanced-usage.html#ssl-warnings
  warnings.warn(

2023-12-12 16:37:57.803 WARNING (SyncWorker_5) [py.warnings] /usr/local/lib/python3.11/site-packages/urllib3/connectionpool.py:1061: InsecureRequestWarning: Unverified HTTPS request is being made to host '192.168.69.103'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/1.26.x/advanced-usage.html#ssl-warnings
  warnings.warn(

2023-12-12 16:37:58.944 WARNING (SyncWorker_2) [py.warnings] /usr/local/lib/python3.11/site-packages/urllib3/connectionpool.py:1061: InsecureRequestWarning: Unverified HTTPS request is being made to host '192.168.69.103'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/1.26.x/advanced-usage.html#ssl-warnings
  warnings.warn(

2023-12-12 16:37:58.951 WARNING (SyncWorker_2) [py.warnings] /usr/local/lib/python3.11/site-packages/urllib3/connectionpool.py:1061: InsecureRequestWarning: Unverified HTTPS request is being made to host '192.168.69.103'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/1.26.x/advanced-usage.html#ssl-warnings
  warnings.warn(

2023-12-12 16:38:14.170 WARNING (MainThread) [homeassistant.bootstrap] Waiting on integrations to complete setup: mqtt
2023-12-12 16:38:20.805 WARNING (SyncWorker_5) [py.warnings] /usr/local/lib/python3.11/site-packages/urllib3/connectionpool.py:1061: InsecureRequestWarning: Unverified HTTPS request is being made to host '192.168.69.103'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/1.26.x/advanced-usage.html#ssl-warnings
  warnings.warn(

2023-12-12 16:38:20.913 WARNING (SyncWorker_5) [py.warnings] /usr/local/lib/python3.11/site-packages/urllib3/connectionpool.py:1061: InsecureRequestWarning: Unverified HTTPS request is being made to host '192.168.69.103'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/1.26.x/advanced-usage.html#ssl-warnings
  warnings.warn(

2023-12-12 16:38:20.967 WARNING (SyncWorker_5) [py.warnings] /usr/local/lib/python3.11/site-packages/urllib3/connectionpool.py:1061: InsecureRequestWarning: Unverified HTTPS request is being made to host '192.168.69.103'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/1.26.x/advanced-usage.html#ssl-warnings
  warnings.warn(

2023-12-12 16:38:20.980 WARNING (SyncWorker_5) [py.warnings] /usr/local/lib/python3.11/site-packages/urllib3/connectionpool.py:1061: InsecureRequestWarning: Unverified HTTPS request is being made to host '192.168.69.103'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/1.26.x/advanced-usage.html#ssl-warnings
  warnings.warn(

2023-12-12 16:38:21.010 WARNING (SyncWorker_5) [py.warnings] /usr/local/lib/python3.11/site-packages/urllib3/connectionpool.py:1061: InsecureRequestWarning: Unverified HTTPS request is being made to host '192.168.69.103'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/1.26.x/advanced-usage.html#ssl-warnings
  warnings.warn(

2023-12-12 16:38:21.222 WARNING (SyncWorker_5) [py.warnings] /usr/local/lib/python3.11/site-packages/urllib3/connectionpool.py:1061: InsecureRequestWarning: Unverified HTTPS request is being made to host '192.168.69.103'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/1.26.x/advanced-usage.html#ssl-warnings
  warnings.warn(

2023-12-12 16:38:21.229 WARNING (SyncWorker_5) [py.warnings] /usr/local/lib/python3.11/site-packages/urllib3/connectionpool.py:1061: InsecureRequestWarning: Unverified HTTPS request is being made to host '192.168.69.103'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/1.26.x/advanced-usage.html#ssl-warnings
  warnings.warn(

2023-12-12 16:38:21.242 WARNING (SyncWorker_5) [py.warnings] /usr/local/lib/python3.11/site-packages/urllib3/connectionpool.py:1061: InsecureRequestWarning: Unverified HTTPS request is being made to host '192.168.69.103'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/1.26.x/advanced-usage.html#ssl-warnings
  warnings.warn(

2023-12-12 16:38:21.258 WARNING (SyncWorker_5) [py.warnings] /usr/local/lib/python3.11/site-packages/urllib3/connectionpool.py:1061: InsecureRequestWarning: Unverified HTTPS request is being made to host '192.168.69.103'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/1.26.x/advanced-usage.html#ssl-warnings
  warnings.warn(

2023-12-12 16:38:21.979 ERROR (MainThread) [homeassistant.helpers.integration_platform] Error processing platform plex_assistant.intent
Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/helpers/integration_platform.py", line 61, in _async_process_single_integration_platform_component
    await integration_platform.process_platform(hass, component_name, platform)
  File "/usr/src/homeassistant/homeassistant/components/intent/__init__.py", line 254, in _async_process_intent
    await platform.async_setup_intents(hass)
  File "/config/custom_components/plex_assistant/intent.py", line 9, in async_setup_intents
    hass.components.conversation.async_register(
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/src/homeassistant/homeassistant/loader.py", line 1067, in __getattr__
    value = getattr(self._module, attr)
            ^^^^^^^^^^^^^^^^^^^^^^^^^^^
AttributeError: module 'homeassistant.components.conversation' has no attribute 'async_register'
2023-12-12 16:38:31.572 WARNING (MainThread) [homeassistant.components.media_player] Setup of media_player platform dlna_dmr is taking over 10 seconds.
2023-12-12 16:38:31.874 ERROR (MainThread) [homeassistant.helpers.event] Error while processing template: Template<template=({{ (states.sensor.desk_light_button_device_temperature.state | float - 8 ) | round(0) }}) renders=2>
Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/helpers/template.py", line 1926, in forgiving_float_filter
    return float(value)
           ^^^^^^^^^^^^
ValueError: could not convert string to float: 'unavailable'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/helpers/template.py", line 569, in async_render
    render_result = _render_with_context(self.template, compiled, **kwargs)
                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/src/homeassistant/homeassistant/helpers/template.py", line 2247, in _render_with_context
    return template.render(**kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/jinja2/environment.py", line 1301, in render
    self.environment.handle_exception()
  File "/usr/local/lib/python3.11/site-packages/jinja2/environment.py", line 936, in handle_exception
    raise rewrite_traceback_stack(source=source)
  File "<template>", line 1, in top-level template code
  File "/usr/src/homeassistant/homeassistant/helpers/template.py", line 1929, in forgiving_float_filter
    raise_no_default("float", value)
  File "/usr/src/homeassistant/homeassistant/helpers/template.py", line 1627, in raise_no_default
    raise ValueError(
ValueError: Template error: float got invalid input 'unavailable' when rendering template '{{ (states.sensor.desk_light_button_device_temperature.state | float - 8 ) | round(0) }}' but no default was specified

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/helpers/template.py", line 695, in async_render_to_info
    render_info._result = self.async_render(
                          ^^^^^^^^^^^^^^^^^^
  File "/usr/src/homeassistant/homeassistant/helpers/template.py", line 571, in async_render
    raise TemplateError(err) from err
homeassistant.exceptions.TemplateError: ValueError: Template error: float got invalid input 'unavailable' when rendering template '{{ (states.sensor.desk_light_button_device_temperature.state | float - 8 ) | round(0) }}' but no default was specified
2023-12-12 16:38:31.892 ERROR (MainThread) [homeassistant.components.template.template_entity] TemplateError('ValueError: Template error: float got invalid input 'unavailable' when rendering template '{{ (states.sensor.desk_light_button_device_temperature.state | float - 8 ) | round(0) }}' but no default was specified') while processing template 'Template<template=({{ (states.sensor.desk_light_button_device_temperature.state | float - 8 ) | round(0) }}) renders=4>' for attribute '_attr_native_value' in entity 'sensor.desk_temp_adj'
2023-12-12 16:38:31.894 ERROR (MainThread) [homeassistant.helpers.event] Error while processing template: Template<template=({{ (states.sensor.front_door_sensor_device_temperature.state | float - 3 ) | round(0) }}) renders=2>
Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/helpers/template.py", line 1926, in forgiving_float_filter
    return float(value)
           ^^^^^^^^^^^^
ValueError: could not convert string to float: 'unavailable'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/helpers/template.py", line 569, in async_render
    render_result = _render_with_context(self.template, compiled, **kwargs)
                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/src/homeassistant/homeassistant/helpers/template.py", line 2247, in _render_with_context
    return template.render(**kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/jinja2/environment.py", line 1301, in render
    self.environment.handle_exception()
  File "/usr/local/lib/python3.11/site-packages/jinja2/environment.py", line 936, in handle_exception
    raise rewrite_traceback_stack(source=source)
  File "<template>", line 1, in top-level template code
  File "/usr/src/homeassistant/homeassistant/helpers/template.py", line 1929, in forgiving_float_filter
    raise_no_default("float", value)
  File "/usr/src/homeassistant/homeassistant/helpers/template.py", line 1627, in raise_no_default
    raise ValueError(
ValueError: Template error: float got invalid input 'unavailable' when rendering template '{{ (states.sensor.front_door_sensor_device_temperature.state | float - 3 ) | round(0) }}' but no default was specified

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/helpers/template.py", line 695, in async_render_to_info
    render_info._result = self.async_render(
                          ^^^^^^^^^^^^^^^^^^
  File "/usr/src/homeassistant/homeassistant/helpers/template.py", line 571, in async_render
    raise TemplateError(err) from err
homeassistant.exceptions.TemplateError: ValueError: Template error: float got invalid input 'unavailable' when rendering template '{{ (states.sensor.front_door_sensor_device_temperature.state | float - 3 ) | round(0) }}' but no default was specified
2023-12-12 16:38:31.903 ERROR (MainThread) [homeassistant.components.template.template_entity] TemplateError('ValueError: Template error: float got invalid input 'unavailable' when rendering template '{{ (states.sensor.front_door_sensor_device_temperature.state | float - 3 ) | round(0) }}' but no default was specified') while processing template 'Template<template=({{ (states.sensor.front_door_sensor_device_temperature.state | float - 3 ) | round(0) }}) renders=4>' for attribute '_attr_native_value' in entity 'sensor.front_door_sensor_device_temperature_adj'
2023-12-12 16:38:31.905 ERROR (MainThread) [homeassistant.helpers.event] Error while processing template: Template<template=({{ (states.sensor.living_room_window_sensor_device_temperature.state | float - 4 ) | round(0) }}) renders=2>
Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/helpers/template.py", line 1926, in forgiving_float_filter
    return float(value)
           ^^^^^^^^^^^^
ValueError: could not convert string to float: 'unavailable'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/helpers/template.py", line 569, in async_render
    render_result = _render_with_context(self.template, compiled, **kwargs)
                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/src/homeassistant/homeassistant/helpers/template.py", line 2247, in _render_with_context
    return template.render(**kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/jinja2/environment.py", line 1301, in render
    self.environment.handle_exception()
  File "/usr/local/lib/python3.11/site-packages/jinja2/environment.py", line 936, in handle_exception
    raise rewrite_traceback_stack(source=source)
  File "<template>", line 1, in top-level template code
  File "/usr/src/homeassistant/homeassistant/helpers/template.py", line 1929, in forgiving_float_filter
    raise_no_default("float", value)
  File "/usr/src/homeassistant/homeassistant/helpers/template.py", line 1627, in raise_no_default
    raise ValueError(
ValueError: Template error: float got invalid input 'unavailable' when rendering template '{{ (states.sensor.living_room_window_sensor_device_temperature.state | float - 4 ) | round(0) }}' but no default was specified

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/helpers/template.py", line 695, in async_render_to_info
    render_info._result = self.async_render(
                          ^^^^^^^^^^^^^^^^^^
  File "/usr/src/homeassistant/homeassistant/helpers/template.py", line 571, in async_render
    raise TemplateError(err) from err
homeassistant.exceptions.TemplateError: ValueError: Template error: float got invalid input 'unavailable' when rendering template '{{ (states.sensor.living_room_window_sensor_device_temperature.state | float - 4 ) | round(0) }}' but no default was specified
2023-12-12 16:38:31.913 ERROR (MainThread) [homeassistant.components.template.template_entity] TemplateError('ValueError: Template error: float got invalid input 'unavailable' when rendering template '{{ (states.sensor.living_room_window_sensor_device_temperature.state | float - 4 ) | round(0) }}' but no default was specified') while processing template 'Template<template=({{ (states.sensor.living_room_window_sensor_device_temperature.state | float - 4 ) | round(0) }}) renders=4>' for attribute '_attr_native_value' in entity 'sensor.living_room_window_sensor_device_temperature_adj'
2023-12-12 16:38:31.916 ERROR (MainThread) [homeassistant.helpers.template] Template variable error: 'None' has no attribute 'attributes' when rendering '{{ states.vacuum.hombot.attributes.battery_level }}'
2023-12-12 16:38:31.916 ERROR (MainThread) [homeassistant.helpers.event] Error while processing template: Template<template=({{ states.vacuum.hombot.attributes.battery_level }}) renders=2>
Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/helpers/template.py", line 569, in async_render
    render_result = _render_with_context(self.template, compiled, **kwargs)
                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/src/homeassistant/homeassistant/helpers/template.py", line 2247, in _render_with_context
    return template.render(**kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/jinja2/environment.py", line 1301, in render
    self.environment.handle_exception()
  File "/usr/local/lib/python3.11/site-packages/jinja2/environment.py", line 936, in handle_exception
    raise rewrite_traceback_stack(source=source)
  File "<template>", line 1, in top-level template code
  File "/usr/local/lib/python3.11/site-packages/jinja2/sandbox.py", line 326, in getattr
    value = getattr(obj, attribute)
            ^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/src/homeassistant/homeassistant/helpers/template.py", line 2282, in _fail_with_undefined_error
    raise ex
  File "/usr/src/homeassistant/homeassistant/helpers/template.py", line 2279, in _fail_with_undefined_error
    return super()._fail_with_undefined_error(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
jinja2.exceptions.UndefinedError: 'None' has no attribute 'attributes'

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/helpers/template.py", line 695, in async_render_to_info
    render_info._result = self.async_render(
                          ^^^^^^^^^^^^^^^^^^
  File "/usr/src/homeassistant/homeassistant/helpers/template.py", line 571, in async_render
    raise TemplateError(err) from err
homeassistant.exceptions.TemplateError: UndefinedError: 'None' has no attribute 'attributes'
2023-12-12 16:38:31.926 ERROR (MainThread) [homeassistant.helpers.template] Template variable error: 'None' has no attribute 'attributes' when rendering '{{ states.vacuum.hombot.attributes.battery_icon }}'
2023-12-12 16:38:31.927 ERROR (MainThread) [homeassistant.helpers.event] Error while processing template: Template<template=({{ states.vacuum.hombot.attributes.battery_icon }}) renders=3>
Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/helpers/template.py", line 569, in async_render
    render_result = _render_with_context(self.template, compiled, **kwargs)
                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/src/homeassistant/homeassistant/helpers/template.py", line 2247, in _render_with_context
    return template.render(**kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/jinja2/environment.py", line 1301, in render
    self.environment.handle_exception()
  File "/usr/local/lib/python3.11/site-packages/jinja2/environment.py", line 936, in handle_exception
    raise rewrite_traceback_stack(source=source)
  File "<template>", line 1, in top-level template code
  File "/usr/local/lib/python3.11/site-packages/jinja2/sandbox.py", line 326, in getattr
    value = getattr(obj, attribute)
            ^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/src/homeassistant/homeassistant/helpers/template.py", line 2282, in _fail_with_undefined_error
    raise ex
  File "/usr/src/homeassistant/homeassistant/helpers/template.py", line 2279, in _fail_with_undefined_error
    return super()._fail_with_undefined_error(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
jinja2.exceptions.UndefinedError: 'None' has no attribute 'attributes'

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/helpers/template.py", line 695, in async_render_to_info
    render_info._result = self.async_render(
                          ^^^^^^^^^^^^^^^^^^
  File "/usr/src/homeassistant/homeassistant/helpers/template.py", line 571, in async_render
    raise TemplateError(err) from err
homeassistant.exceptions.TemplateError: UndefinedError: 'None' has no attribute 'attributes'
2023-12-12 16:38:31.935 ERROR (MainThread) [homeassistant.helpers.template] Template variable error: 'None' has no attribute 'attributes' when rendering '{{ states.vacuum.hombot.attributes.battery_level }}'
2023-12-12 16:38:31.936 ERROR (MainThread) [homeassistant.helpers.template] Template variable error: 'None' has no attribute 'attributes' when rendering '{{ states.vacuum.hombot.attributes.battery_icon }}'
2023-12-12 16:38:31.937 ERROR (MainThread) [homeassistant.components.template.template_entity] TemplateError('UndefinedError: 'None' has no attribute 'attributes'') while processing template 'Template<template=({{ states.vacuum.hombot.attributes.battery_level }}) renders=4>' for attribute '_attr_native_value' in entity 'sensor.hombot_battery'
2023-12-12 16:38:31.937 ERROR (MainThread) [homeassistant.components.template.template_entity] TemplateError('UndefinedError: 'None' has no attribute 'attributes'') while processing template 'Template<template=({{ states.vacuum.hombot.attributes.battery_icon }}) renders=5>' for attribute '_attr_icon' in entity 'sensor.hombot_battery'
2023-12-12 16:38:31.938 ERROR (MainThread) [homeassistant.helpers.template] Template variable error: 'None' has no attribute 'attributes' when rendering '{{ states.vacuum.hombot.attributes.status }}'
2023-12-12 16:38:31.939 ERROR (MainThread) [homeassistant.helpers.event] Error while processing template: Template<template=({{ states.vacuum.hombot.attributes.status }}) renders=2>
Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/helpers/template.py", line 569, in async_render
    render_result = _render_with_context(self.template, compiled, **kwargs)
                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/src/homeassistant/homeassistant/helpers/template.py", line 2247, in _render_with_context
    return template.render(**kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/jinja2/environment.py", line 1301, in render
    self.environment.handle_exception()
  File "/usr/local/lib/python3.11/site-packages/jinja2/environment.py", line 936, in handle_exception
    raise rewrite_traceback_stack(source=source)
  File "<template>", line 1, in top-level template code
  File "/usr/local/lib/python3.11/site-packages/jinja2/sandbox.py", line 326, in getattr
    value = getattr(obj, attribute)
            ^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/src/homeassistant/homeassistant/helpers/template.py", line 2282, in _fail_with_undefined_error
    raise ex
  File "/usr/src/homeassistant/homeassistant/helpers/template.py", line 2279, in _fail_with_undefined_error
    return super()._fail_with_undefined_error(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
jinja2.exceptions.UndefinedError: 'None' has no attribute 'attributes'

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/helpers/template.py", line 695, in async_render_to_info
    render_info._result = self.async_render(
                          ^^^^^^^^^^^^^^^^^^
  File "/usr/src/homeassistant/homeassistant/helpers/template.py", line 571, in async_render
    raise TemplateError(err) from err
homeassistant.exceptions.TemplateError: UndefinedError: 'None' has no attribute 'attributes'
2023-12-12 16:38:31.947 ERROR (MainThread) [homeassistant.helpers.template] Template variable error: 'None' has no attribute 'attributes' when rendering '{{ states.vacuum.hombot.attributes.status }}'
2023-12-12 16:38:31.948 ERROR (MainThread) [homeassistant.components.template.template_entity] TemplateError('UndefinedError: 'None' has no attribute 'attributes'') while processing template 'Template<template=({{ states.vacuum.hombot.attributes.status }}) renders=4>' for attribute '_attr_native_value' in entity 'sensor.hombot_status'
2023-12-12 16:38:31.953 ERROR (MainThread) [homeassistant.helpers.event] Error while processing template: Template<template=({% if states('sensor.medicine_hours_recent')|int > 23 %}
  {% set hours = states('sensor.medicine_hours_recent')|int - 24 %}
{% else %}
  {% set hours = states('sensor.medicine_hours_recent')|int %}
{% endif %}
  {% set minutes = states('sensor.medicine_minutes_recent')|int %}
{{hours|string + ":" + minutes|string }}) renders=2>
Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/helpers/template.py", line 569, in async_render
    render_result = _render_with_context(self.template, compiled, **kwargs)
                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/src/homeassistant/homeassistant/helpers/template.py", line 2247, in _render_with_context
    return template.render(**kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/jinja2/environment.py", line 1301, in render
    self.environment.handle_exception()
  File "/usr/local/lib/python3.11/site-packages/jinja2/environment.py", line 936, in handle_exception
    raise rewrite_traceback_stack(source=source)
  File "<template>", line 1, in top-level template code
  File "/usr/src/homeassistant/homeassistant/helpers/template.py", line 1945, in forgiving_int_filter
    raise_no_default("int", value)
  File "/usr/src/homeassistant/homeassistant/helpers/template.py", line 1627, in raise_no_default
    raise ValueError(
ValueError: Template error: int got invalid input 'unknown' when rendering template '{% if states('sensor.medicine_hours_recent')|int > 23 %}
  {% set hours = states('sensor.medicine_hours_recent')|int - 24 %}
{% else %}
  {% set hours = states('sensor.medicine_hours_recent')|int %}
{% endif %}
  {% set minutes = states('sensor.medicine_minutes_recent')|int %}
{{hours|string + ":" + minutes|string }}' but no default was specified

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/helpers/template.py", line 695, in async_render_to_info
    render_info._result = self.async_render(
                          ^^^^^^^^^^^^^^^^^^
  File "/usr/src/homeassistant/homeassistant/helpers/template.py", line 571, in async_render
    raise TemplateError(err) from err
homeassistant.exceptions.TemplateError: ValueError: Template error: int got invalid input 'unknown' when rendering template '{% if states('sensor.medicine_hours_recent')|int > 23 %}
  {% set hours = states('sensor.medicine_hours_recent')|int - 24 %}
{% else %}
  {% set hours = states('sensor.medicine_hours_recent')|int %}
{% endif %}
  {% set minutes = states('sensor.medicine_minutes_recent')|int %}
{{hours|string + ":" + minutes|string }}' but no default was specified
2023-12-12 16:38:31.964 ERROR (MainThread) [homeassistant.components.template.template_entity] TemplateError('ValueError: Template error: int got invalid input 'unknown' when rendering template '{% if states('sensor.medicine_hours_recent')|int > 23 %}
  {% set hours = states('sensor.medicine_hours_recent')|int - 24 %}
{% else %}
  {% set hours = states('sensor.medicine_hours_recent')|int %}
{% endif %}
  {% set minutes = states('sensor.medicine_minutes_recent')|int %}
{{hours|string + ":" + minutes|string }}' but no default was specified') while processing template 'Template<template=({% if states('sensor.medicine_hours_recent')|int > 23 %}
  {% set hours = states('sensor.medicine_hours_recent')|int - 24 %}
{% else %}
  {% set hours = states('sensor.medicine_hours_recent')|int %}
{% endif %}
  {% set minutes = states('sensor.medicine_minutes_recent')|int %}
{{hours|string + ":" + minutes|string }}) renders=4>' for attribute '_attr_native_value' in entity 'sensor.medicine_time_recent'
2023-12-12 16:38:32.055 WARNING (MainThread) [homeassistant.components.homeassistant.triggers.numeric_state] Error initializing 'Living rooms - lights on when blinds close' trigger: In 'numeric_state' condition: entity cover.living_room_blinds state 'open' cannot be processed as a number
2023-12-12 16:38:32.068 WARNING (MainThread) [homeassistant.components.homeassistant.triggers.numeric_state] Error initializing 'Bedroom - lights off when blinds open' trigger: In 'numeric_state' condition: entity cover.bedroom_blinds state 'open' cannot be processed as a number
2023-12-12 16:38:36.855 WARNING (MainThread) [custom_components.hacs] You have 'ericpignet/home-assistant-lg_hombot' installed with HACS this repository has been removed from HACS, please consider removing it. Removal reason (Integration is missing a version, and is abandoned.)
2023-12-12 16:38:36.856 WARNING (MainThread) [custom_components.hacs] You have 'ljmerza/ha-our-groceries' installed with HACS this repository has been removed from HACS, please consider removing it. Removal reason (Repository is archived)
2023-12-12 16:38:42.490 WARNING (Recorder) [homeassistant.components.sensor.recorder] The unit of sensor.openweathermap_rain (mm/h) cannot be converted to the unit of previously compiled statistics (mm). Generation of long term statistics will be suppressed unless the unit changes back to mm or a compatible unit. Go to https://my.home-assistant.io/redirect/developer_statistics to fix this
2023-12-12 16:38:42.491 WARNING (Recorder) [homeassistant.components.sensor.recorder] The unit of sensor.openweathermap_snow (mm/h) cannot be converted to the unit of previously compiled statistics (mm). Generation of long term statistics will be suppressed unless the unit changes back to mm or a compatible unit. Go to https://my.home-assistant.io/redirect/developer_statistics to fix this
2023-12-12 16:38:42.501 WARNING (Recorder) [homeassistant.components.sensor.recorder] The unit of sensor.home_precipitation (mm/h) cannot be converted to the unit of previously compiled statistics (mm). Generation of long term statistics will be suppressed unless the unit changes back to mm or a compatible unit. Go to https://my.home-assistant.io/redirect/developer_statistics to fix this
2023-12-12 16:39:07.629 WARNING (MainThread) [homeassistant.helpers.template] Template variable warning: 'title' is undefined when rendering '{{ title }}'
2023-12-12 16:39:07.632 WARNING (MainThread) [homeassistant.helpers.template] Template variable warning: 'title' is undefined when rendering '{{ title }}'
2023-12-12 16:39:07.635 WARNING (MainThread) [homeassistant.helpers.template] Template variable warning: 'title' is undefined when rendering '{{ title }}'
2023-12-12 16:39:07.640 ERROR (MainThread) [homeassistant.components.script.notify_people_who_are_present] Notify people who are home: If at step 2: Error executing script. Service not found for call_service at pos 1: Service notify.mobile_app_user2_phone not found.
2023-12-12 16:39:07.642 ERROR (MainThread) [homeassistant.components.script.notify_people_who_are_present] Notify people who are home: If at step 2: Error executing script. Service not found for call_service at pos 1: Service notify.mobile_app_user2_phone not found.
2023-12-12 16:39:07.643 ERROR (MainThread) [homeassistant.components.script.notify_people_who_are_present] Notify people who are home: If at step 2: Error executing script. Service not found for call_service at pos 1: Service notify.mobile_app_user2_phone not found.
2023-12-12 16:39:07.647 ERROR (MainThread) [homeassistant.components.script.notify_people_who_are_present] Notify people who are home: Error executing script. Service not found for if at pos 2: Service notify.mobile_app_user2_phone not found.
2023-12-12 16:39:07.648 ERROR (MainThread) [homeassistant.components.script.notify_people_who_are_present] Notify people who are home: Error executing script. Service not found for if at pos 2: Service notify.mobile_app_user2_phone not found.
2023-12-12 16:39:07.649 ERROR (MainThread) [homeassistant.components.script.notify_people_who_are_present] Notify people who are home: Error executing script. Service not found for if at pos 2: Service notify.mobile_app_user2_phone not found.
2023-12-12 16:39:07.656 ERROR (MainThread) [homeassistant.components.automation.all_rooms_rain_alert_when_window_open] All rooms - Rain alert when window open: If at step 4: Error executing script. Service not found for call_service at pos 1: Service notify.mobile_app_user2_phone not found.
2023-12-12 16:39:07.657 ERROR (MainThread) [homeassistant.components.automation.all_rooms_rain_alert_when_window_open] All rooms - Rain alert when window open: If at step 4: Error executing script. Service not found for call_service at pos 1: Service notify.mobile_app_user2_phone not found.
2023-12-12 16:39:07.659 ERROR (MainThread) [homeassistant.components.automation.all_rooms_rain_alert_when_window_open] All rooms - Rain alert when window open: If at step 4: Error executing script. Service not found for call_service at pos 1: Service notify.mobile_app_user2_phone not found.
2023-12-12 16:39:07.665 ERROR (MainThread) [homeassistant.components.automation.all_rooms_rain_alert_when_window_open] All rooms - Rain alert when window open: Error executing script. Service not found for if at pos 4: Service notify.mobile_app_user2_phone not found.
2023-12-12 16:39:07.666 ERROR (MainThread) [homeassistant.components.automation.all_rooms_rain_alert_when_window_open] All rooms - Rain alert when window open: Error executing script. Service not found for if at pos 4: Service notify.mobile_app_user2_phone not found.
2023-12-12 16:39:07.667 ERROR (MainThread) [homeassistant.components.automation.all_rooms_rain_alert_when_window_open] All rooms - Rain alert when window open: Error executing script. Service not found for if at pos 4: Service notify.mobile_app_user2_phone not found.
2023-12-12 16:40:07.554 WARNING (MainThread) [homeassistant.components.automation.hassio_control_washer_input_booleans_for_google] *HASSIO - Control washer input booleans for Google: Already running
2023-12-12 16:41:28.695 ERROR (SyncWorker_3) [homeassistant.components.command_line.utils] Command failed (with return code 1): python3 -c "import requests; import json; import random; dataRequest = requests.get('https://gpsgadget.buienradar.nl/data/raintext?lat=51.83&lon=5.86&c='+str(random.randint(0,999999999999999)) ).text; dataRequest = dataRequest.replace('\r\n',' '); data = '{\"data\":\"'+dataRequest+'\"}';    pri(data);"
2023-12-12 16:41:28.697 WARNING (MainThread) [homeassistant.components.command_line] Empty reply found when expecting JSON data
2023-12-12 16:42:07.605 WARNING (MainThread) [homeassistant.helpers.script] Error in 'if[0]' evaluation: In 'numeric_state':
  In 'numeric_state' condition: unknown entity sensor.humidity_11
2023-12-12 16:45:28.597 ERROR (SyncWorker_9) [homeassistant.components.command_line.utils] Command failed (with return code 1): python3 -c "import requests; import json; import random; dataRequest = requests.get('https://gpsgadget.buienradar.nl/data/raintext?lat=51.83&lon=5.86&c='+str(random.randint(0,999999999999999)) ).text; dataRequest = dataRequest.replace('\r\n',' '); data = '{\"data\":\"'+dataRequest+'\"}';    pri(data);"
2023-12-12 16:45:28.599 WARNING (MainThread) [homeassistant.components.command_line] Empty reply found when expecting JSON data
2023-12-12 16:49:28.653 ERROR (SyncWorker_2) [homeassistant.components.command_line.utils] Command failed (with return code 1): python3 -c "import requests; import json; import random; dataRequest = requests.get('https://gpsgadget.buienradar.nl/data/raintext?lat=51.83&lon=5.86&c='+str(random.randint(0,999999999999999)) ).text; dataRequest = dataRequest.replace('\r\n',' '); data = '{\"data\":\"'+dataRequest+'\"}';    pri(data);"
2023-12-12 16:49:28.654 WARNING (MainThread) [homeassistant.components.command_line] Empty reply found when expecting JSON data
2023-12-12 16:53:28.760 ERROR (SyncWorker_1) [homeassistant.components.command_line.utils] Command failed (with return code 1): python3 -c "import requests; import json; import random; dataRequest = requests.get('https://gpsgadget.buienradar.nl/data/raintext?lat=51.83&lon=5.86&c='+str(random.randint(0,999999999999999)) ).text; dataRequest = dataRequest.replace('\r\n',' '); data = '{\"data\":\"'+dataRequest+'\"}';    pri(data);"
2023-12-12 16:53:28.761 WARNING (MainThread) [homeassistant.components.command_line] Empty reply found when expecting JSON data
2023-12-12 16:57:29.491 ERROR (SyncWorker_3) [homeassistant.components.command_line.utils] Command failed (with return code 1): python3 -c "import requests; import json; import random; dataRequest = requests.get('https://gpsgadget.buienradar.nl/data/raintext?lat=51.83&lon=5.86&c='+str(random.randint(0,999999999999999)) ).text; dataRequest = dataRequest.replace('\r\n',' '); data = '{\"data\":\"'+dataRequest+'\"}';    pri(data);"
2023-12-12 16:57:29.492 WARNING (MainThread) [homeassistant.components.command_line] Empty reply found when expecting JSON data
2023-12-12 17:01:29.495 ERROR (SyncWorker_5) [homeassistant.components.command_line.utils] Command failed (with return code 1): python3 -c "import requests; import json; import random; dataRequest = requests.get('https://gpsgadget.buienradar.nl/data/raintext?lat=51.83&lon=5.86&c='+str(random.randint(0,999999999999999)) ).text; dataRequest = dataRequest.replace('\r\n',' '); data = '{\"data\":\"'+dataRequest+'\"}';    pri(data);"
2023-12-12 17:01:29.496 WARNING (MainThread) [homeassistant.components.command_line] Empty reply found when expecting JSON data
2023-12-12 17:05:29.499 ERROR (SyncWorker_1) [homeassistant.components.command_line.utils] Command failed (with return code 1): python3 -c "import requests; import json; import random; dataRequest = requests.get('https://gpsgadget.buienradar.nl/data/raintext?lat=51.83&lon=5.86&c='+str(random.randint(0,999999999999999)) ).text; dataRequest = dataRequest.replace('\r\n',' '); data = '{\"data\":\"'+dataRequest+'\"}';    pri(data);"
2023-12-12 17:05:29.501 WARNING (MainThread) [homeassistant.components.command_line] Empty reply found when expecting JSON data
2023-12-12 17:09:29.658 ERROR (SyncWorker_8) [homeassistant.components.command_line.utils] Command failed (with return code 1): python3 -c "import requests; import json; import random; dataRequest = requests.get('https://gpsgadget.buienradar.nl/data/raintext?lat=51.83&lon=5.86&c='+str(random.randint(0,999999999999999)) ).text; dataRequest = dataRequest.replace('\r\n',' '); data = '{\"data\":\"'+dataRequest+'\"}';    pri(data);"
2023-12-12 17:09:29.660 WARNING (MainThread) [homeassistant.components.command_line] Empty reply found when expecting JSON data
2023-12-12 17:13:29.532 ERROR (SyncWorker_1) [homeassistant.components.command_line.utils] Command failed (with return code 1): python3 -c "import requests; import json; import random; dataRequest = requests.get('https://gpsgadget.buienradar.nl/data/raintext?lat=51.83&lon=5.86&c='+str(random.randint(0,999999999999999)) ).text; dataRequest = dataRequest.replace('\r\n',' '); data = '{\"data\":\"'+dataRequest+'\"}';    pri(data);"
2023-12-12 17:13:29.534 WARNING (MainThread) [homeassistant.components.command_line] Empty reply found when expecting JSON data
2023-12-12 17:17:29.728 ERROR (SyncWorker_4) [homeassistant.components.command_line.utils] Command failed (with return code 1): python3 -c "import requests; import json; import random; dataRequest = requests.get('https://gpsgadget.buienradar.nl/data/raintext?lat=51.83&lon=5.86&c='+str(random.randint(0,999999999999999)) ).text; dataRequest = dataRequest.replace('\r\n',' '); data = '{\"data\":\"'+dataRequest+'\"}';    pri(data);"
2023-12-12 17:17:29.730 WARNING (MainThread) [homeassistant.components.command_line] Empty reply found when expecting JSON data
2023-12-12 17:21:29.750 ERROR (SyncWorker_9) [homeassistant.components.command_line.utils] Command failed (with return code 1): python3 -c "import requests; import json; import random; dataRequest = requests.get('https://gpsgadget.buienradar.nl/data/raintext?lat=51.83&lon=5.86&c='+str(random.randint(0,999999999999999)) ).text; dataRequest = dataRequest.replace('\r\n',' '); data = '{\"data\":\"'+dataRequest+'\"}';    pri(data);"
2023-12-12 17:21:29.751 WARNING (MainThread) [homeassistant.components.command_line] Empty reply found when expecting JSON data
2023-12-12 17:23:38.782 WARNING (MainThread) [homeassistant.helpers.template] Template variable warning: 'title' is undefined when rendering '{{ title }}'
2023-12-12 17:23:39.141 WARNING (MainThread) [homeassistant.helpers.template] Template variable warning: 'title' is undefined when rendering '{{ title }}'
2023-12-12 17:23:39.143 ERROR (MainThread) [homeassistant.components.script.notify_people_who_are_present] Notify people who are home: If at step 2: Error executing script. Service not found for call_service at pos 1: Service notify.mobile_app_user2_phone not found.
2023-12-12 17:23:39.144 ERROR (MainThread) [homeassistant.components.script.notify_people_who_are_present] Notify people who are home: Error executing script. Service not found for if at pos 2: Service notify.mobile_app_user2_phone not found.
2023-12-12 17:23:39.145 ERROR (MainThread) [homeassistant.components.automation.all_rooms_rain_alert_when_window_open] All rooms - Rain alert when window open: If at step 4: Error executing script. Service not found for call_service at pos 1: Service notify.mobile_app_user2_phone not found.
2023-12-12 17:23:39.146 ERROR (MainThread) [homeassistant.components.automation.all_rooms_rain_alert_when_window_open] All rooms - Rain alert when window open: Error executing script. Service not found for if at pos 4: Service notify.mobile_app_user2_phone not found.
2023-12-12 17:25:29.711 ERROR (SyncWorker_3) [homeassistant.components.command_line.utils] Command failed (with return code 1): python3 -c "import requests; import json; import random; dataRequest = requests.get('https://gpsgadget.buienradar.nl/data/raintext?lat=51.83&lon=5.86&c='+str(random.randint(0,999999999999999)) ).text; dataRequest = dataRequest.replace('\r\n',' '); data = '{\"data\":\"'+dataRequest+'\"}';    pri(data);"
2023-12-12 17:25:29.713 WARNING (MainThread) [homeassistant.components.command_line] Empty reply found when expecting JSON data
2023-12-12 17:25:50.045 WARNING (MainThread) [homeassistant.helpers.template] Template variable warning: 'title' is undefined when rendering '{{ title }}'
2023-12-12 17:25:50.421 WARNING (MainThread) [homeassistant.helpers.template] Template variable warning: 'title' is undefined when rendering '{{ title }}'
2023-12-12 17:25:50.425 ERROR (MainThread) [homeassistant.components.script.notify_people_who_are_present] Notify people who are home: If at step 2: Error executing script. Service not found for call_service at pos 1: Service notify.mobile_app_user2_phone not found.
2023-12-12 17:25:50.426 ERROR (MainThread) [homeassistant.components.script.notify_people_who_are_present] Notify people who are home: Error executing script. Service not found for if at pos 2: Service notify.mobile_app_user2_phone not found.
2023-12-12 17:25:50.428 ERROR (MainThread) [homeassistant.components.automation.all_rooms_rain_alert_when_window_open] All rooms - Rain alert when window open: If at step 4: Error executing script. Service not found for call_service at pos 1: Service notify.mobile_app_user2_phone not found.
2023-12-12 17:25:50.429 ERROR (MainThread) [homeassistant.components.automation.all_rooms_rain_alert_when_window_open] All rooms - Rain alert when window open: Error executing script. Service not found for if at pos 4: Service notify.mobile_app_user2_phone not found.
2023-12-12 17:27:33.715 WARNING (MainThread) [homeassistant.components.homeassistant.triggers.numeric_state] Error in 'Living rooms - lights on when blinds close' trigger: In 'numeric_state' condition: entity cover.living_room_blinds state 'open' cannot be processed as a number
2023-12-12 17:27:34.665 WARNING (MainThread) [homeassistant.components.homeassistant.triggers.numeric_state] Error in 'Living rooms - lights on when blinds close' trigger: In 'numeric_state' condition: entity cover.living_room_blinds state 'open' cannot be processed as a number
2023-12-12 17:27:35.637 WARNING (MainThread) [homeassistant.components.homeassistant.triggers.numeric_state] Error in 'Living rooms - lights on when blinds close' trigger: In 'numeric_state' condition: entity cover.living_room_blinds state 'open' cannot be processed as a number
2023-12-12 17:27:36.583 WARNING (MainThread) [homeassistant.components.homeassistant.triggers.numeric_state] Error in 'Living rooms - lights on when blinds close' trigger: In 'numeric_state' condition: entity cover.living_room_blinds state 'open' cannot be processed as a number
2023-12-12 17:27:37.523 WARNING (MainThread) [homeassistant.components.homeassistant.triggers.numeric_state] Error in 'Living rooms - lights on when blinds close' trigger: In 'numeric_state' condition: entity cover.living_room_blinds state 'open' cannot be processed as a number
2023-12-12 17:27:38.462 WARNING (MainThread) [homeassistant.components.homeassistant.triggers.numeric_state] Error in 'Living rooms - lights on when blinds close' trigger: In 'numeric_state' condition: entity cover.living_room_blinds state 'open' cannot be processed as a number
2023-12-12 17:27:39.434 WARNING (MainThread) [homeassistant.components.homeassistant.triggers.numeric_state] Error in 'Living rooms - lights on when blinds close' trigger: In 'numeric_state' condition: entity cover.living_room_blinds state 'open' cannot be processed as a number
2023-12-12 17:27:40.373 WARNING (MainThread) [homeassistant.components.homeassistant.triggers.numeric_state] Error in 'Living rooms - lights on when blinds close' trigger: In 'numeric_state' condition: entity cover.living_room_blinds state 'open' cannot be processed as a number
2023-12-12 17:27:41.281 WARNING (MainThread) [homeassistant.components.homeassistant.triggers.numeric_state] Error in 'Living rooms - lights on when blinds close' trigger: In 'numeric_state' condition: entity cover.living_room_blinds state 'open' cannot be processed as a number
2023-12-12 17:27:42.270 WARNING (MainThread) [homeassistant.components.homeassistant.triggers.numeric_state] Error in 'Living rooms - lights on when blinds close' trigger: In 'numeric_state' condition: entity cover.living_room_blinds state 'open' cannot be processed as a number
2023-12-12 17:27:43.210 WARNING (MainThread) [homeassistant.components.homeassistant.triggers.numeric_state] Error in 'Living rooms - lights on when blinds close' trigger: In 'numeric_state' condition: entity cover.living_room_blinds state 'open' cannot be processed as a number
2023-12-12 17:27:44.164 WARNING (MainThread) [homeassistant.components.homeassistant.triggers.numeric_state] Error in 'Living rooms - lights on when blinds close' trigger: In 'numeric_state' condition: entity cover.living_room_blinds state 'open' cannot be processed as a number
2023-12-12 17:27:45.084 WARNING (MainThread) [homeassistant.components.homeassistant.triggers.numeric_state] Error in 'Living rooms - lights on when blinds close' trigger: In 'numeric_state' condition: entity cover.living_room_blinds state 'open' cannot be processed as a number
2023-12-12 17:27:46.057 WARNING (MainThread) [homeassistant.components.homeassistant.triggers.numeric_state] Error in 'Living rooms - lights on when blinds close' trigger: In 'numeric_state' condition: entity cover.living_room_blinds state 'open' cannot be processed as a number
2023-12-12 17:27:47.002 WARNING (MainThread) [homeassistant.components.homeassistant.triggers.numeric_state] Error in 'Living rooms - lights on when blinds close' trigger: In 'numeric_state' condition: entity cover.living_room_blinds state 'open' cannot be processed as a number
2023-12-12 17:27:47.919 WARNING (MainThread) [homeassistant.components.homeassistant.triggers.numeric_state] Error in 'Living rooms - lights on when blinds close' trigger: In 'numeric_state' condition: entity cover.living_room_blinds state 'open' cannot be processed as a number
2023-12-12 17:27:48.886 WARNING (MainThread) [homeassistant.components.homeassistant.triggers.numeric_state] Error in 'Living rooms - lights on when blinds close' trigger: In 'numeric_state' condition: entity cover.living_room_blinds state 'open' cannot be processed as a number
2023-12-12 17:27:49.834 WARNING (MainThread) [homeassistant.components.homeassistant.triggers.numeric_state] Error in 'Living rooms - lights on when blinds close' trigger: In 'numeric_state' condition: entity cover.living_room_blinds state 'open' cannot be processed as a number
2023-12-12 17:27:50.802 WARNING (MainThread) [homeassistant.components.homeassistant.triggers.numeric_state] Error in 'Living rooms - lights on when blinds close' trigger: In 'numeric_state' condition: entity cover.living_room_blinds state 'open' cannot be processed as a number
2023-12-12 17:27:51.751 WARNING (MainThread) [homeassistant.components.homeassistant.triggers.numeric_state] Error in 'Living rooms - lights on when blinds close' trigger: In 'numeric_state' condition: entity cover.living_room_blinds state 'open' cannot be processed as a number
2023-12-12 17:27:52.661 WARNING (MainThread) [homeassistant.components.homeassistant.triggers.numeric_state] Error in 'Living rooms - lights on when blinds close' trigger: In 'numeric_state' condition: entity cover.living_room_blinds state 'open' cannot be processed as a number
2023-12-12 17:27:53.657 WARNING (MainThread) [homeassistant.components.homeassistant.triggers.numeric_state] Error in 'Living rooms - lights on when blinds close' trigger: In 'numeric_state' condition: entity cover.living_room_blinds state 'open' cannot be processed as a number
2023-12-12 17:27:54.578 WARNING (MainThread) [homeassistant.components.homeassistant.triggers.numeric_state] Error in 'Living rooms - lights on when blinds close' trigger: In 'numeric_state' condition: entity cover.living_room_blinds state 'open' cannot be processed as a number
2023-12-12 17:27:55.553 WARNING (MainThread) [homeassistant.components.homeassistant.triggers.numeric_state] Error in 'Living rooms - lights on when blinds close' trigger: In 'numeric_state' condition: entity cover.living_room_blinds state 'open' cannot be processed as a number
2023-12-12 17:27:56.459 WARNING (MainThread) [homeassistant.components.homeassistant.triggers.numeric_state] Error in 'Living rooms - lights on when blinds close' trigger: In 'numeric_state' condition: entity cover.living_room_blinds state 'open' cannot be processed as a number
2023-12-12 17:27:57.462 WARNING (MainThread) [homeassistant.components.homeassistant.triggers.numeric_state] Error in 'Living rooms - lights on when blinds close' trigger: In 'numeric_state' condition: entity cover.living_room_blinds state 'open' cannot be processed as a number
2023-12-12 17:27:58.384 WARNING (MainThread) [homeassistant.components.homeassistant.triggers.numeric_state] Error in 'Living rooms - lights on when blinds close' trigger: In 'numeric_state' condition: entity cover.living_room_blinds state 'open' cannot be processed as a number
2023-12-12 17:27:59.294 WARNING (MainThread) [homeassistant.components.homeassistant.triggers.numeric_state] Error in 'Living rooms - lights on when blinds close' trigger: In 'numeric_state' condition: entity cover.living_room_blinds state 'open' cannot be processed as a number
2023-12-12 17:28:00.254 WARNING (MainThread) [homeassistant.components.homeassistant.triggers.numeric_state] Error in 'Living rooms - lights on when blinds close' trigger: In 'numeric_state' condition: entity cover.living_room_blinds state 'open' cannot be processed as a number
2023-12-12 17:28:01.224 WARNING (MainThread) [homeassistant.components.homeassistant.triggers.numeric_state] Error in 'Living rooms - lights on when blinds close' trigger: In 'numeric_state' condition: entity cover.living_room_blinds state 'open' cannot be processed as a number
2023-12-12 17:28:02.124 WARNING (MainThread) [homeassistant.components.homeassistant.triggers.numeric_state] Error in 'Living rooms - lights on when blinds close' trigger: In 'numeric_state' condition: entity cover.living_room_blinds state 'open' cannot be processed as a number
2023-12-12 17:28:03.136 WARNING (MainThread) [homeassistant.components.homeassistant.triggers.numeric_state] Error in 'Living rooms - lights on when blinds close' trigger: In 'numeric_state' condition: entity cover.living_room_blinds state 'open' cannot be processed as a number
2023-12-12 17:28:04.052 WARNING (MainThread) [homeassistant.components.homeassistant.triggers.numeric_state] Error in 'Living rooms - lights on when blinds close' trigger: In 'numeric_state' condition: entity cover.living_room_blinds state 'open' cannot be processed as a number
2023-12-12 17:28:05.058 WARNING (MainThread) [homeassistant.components.homeassistant.triggers.numeric_state] Error in 'Living rooms - lights on when blinds close' trigger: In 'numeric_state' condition: entity cover.living_room_blinds state 'open' cannot be processed as a number
2023-12-12 17:28:05.921 WARNING (MainThread) [homeassistant.components.homeassistant.triggers.numeric_state] Error in 'Living rooms - lights on when blinds close' trigger: In 'numeric_state' condition: entity cover.living_room_blinds state 'open' cannot be processed as a number
2023-12-12 17:28:06.888 WARNING (MainThread) [homeassistant.components.homeassistant.triggers.numeric_state] Error in 'Living rooms - lights on when blinds close' trigger: In 'numeric_state' condition: entity cover.living_room_blinds state 'open' cannot be processed as a number
2023-12-12 17:28:07.855 WARNING (MainThread) [homeassistant.components.homeassistant.triggers.numeric_state] Error in 'Living rooms - lights on when blinds close' trigger: In 'numeric_state' condition: entity cover.living_room_blinds state 'open' cannot be processed as a number
2023-12-12 17:28:07.982 WARNING (MainThread) [homeassistant.components.homeassistant.triggers.numeric_state] Error in 'Bedroom - lights off when blinds open' trigger: In 'numeric_state' condition: entity cover.bedroom_blinds state 'open' cannot be processed as a number
2023-12-12 17:28:07.986 WARNING (MainThread) [homeassistant.components.automation] Error evaluating condition in 'Bedroom - open window detection':
In 'condition':
  In 'state':
    In 'state' condition: unknown entity binary_sensor.openclose_13
2023-12-12 17:28:08.462 WARNING (MainThread) [homeassistant.components.homeassistant.triggers.numeric_state] Error in 'Bedroom - lights off when blinds open' trigger: In 'numeric_state' condition: entity cover.bedroom_blinds state 'open' cannot be processed as a number
2023-12-12 17:28:08.465 WARNING (MainThread) [homeassistant.components.automation] Error evaluating condition in 'Bedroom - open window detection':
In 'condition':
  In 'state':
    In 'state' condition: unknown entity binary_sensor.openclose_13
2023-12-12 17:28:08.765 WARNING (MainThread) [homeassistant.components.homeassistant.triggers.numeric_state] Error in 'Living rooms - lights on when blinds close' trigger: In 'numeric_state' condition: entity cover.living_room_blinds state 'open' cannot be processed as a number
2023-12-12 17:28:08.875 WARNING (MainThread) [homeassistant.components.homeassistant.triggers.numeric_state] Error in 'Bedroom - lights off when blinds open' trigger: In 'numeric_state' condition: entity cover.bedroom_blinds state 'open' cannot be processed as a number
2023-12-12 17:28:08.882 WARNING (MainThread) [homeassistant.components.automation] Error evaluating condition in 'Bedroom - open window detection':
In 'condition':
  In 'state':
    In 'state' condition: unknown entity binary_sensor.openclose_13
2023-12-12 17:28:09.355 WARNING (MainThread) [homeassistant.components.homeassistant.triggers.numeric_state] Error in 'Bedroom - lights off when blinds open' trigger: In 'numeric_state' condition: entity cover.bedroom_blinds state 'open' cannot be processed as a number
2023-12-12 17:28:09.358 WARNING (MainThread) [homeassistant.components.automation] Error evaluating condition in 'Bedroom - open window detection':
In 'condition':
  In 'state':
    In 'state' condition: unknown entity binary_sensor.openclose_13
2023-12-12 17:28:09.732 WARNING (MainThread) [homeassistant.components.homeassistant.triggers.numeric_state] Error in 'Living rooms - lights on when blinds close' trigger: In 'numeric_state' condition: entity cover.living_room_blinds state 'open' cannot be processed as a number
2023-12-12 17:28:09.736 WARNING (MainThread) [homeassistant.components.homeassistant.triggers.numeric_state] Error in 'Bedroom - lights off when blinds open' trigger: In 'numeric_state' condition: entity cover.bedroom_blinds state 'open' cannot be processed as a number
2023-12-12 17:28:09.740 WARNING (MainThread) [homeassistant.components.automation] Error evaluating condition in 'Bedroom - open window detection':
In 'condition':
  In 'state':
    In 'state' condition: unknown entity binary_sensor.openclose_13
2023-12-12 17:28:10.221 WARNING (MainThread) [homeassistant.components.homeassistant.triggers.numeric_state] Error in 'Bedroom - lights off when blinds open' trigger: In 'numeric_state' condition: entity cover.bedroom_blinds state 'open' cannot be processed as a number
2023-12-12 17:28:10.224 WARNING (MainThread) [homeassistant.components.automation] Error evaluating condition in 'Bedroom - open window detection':
In 'condition':
  In 'state':
    In 'state' condition: unknown entity binary_sensor.openclose_13
2023-12-12 17:28:10.675 WARNING (MainThread) [homeassistant.components.homeassistant.triggers.numeric_state] Error in 'Bedroom - lights off when blinds open' trigger: In 'numeric_state' condition: entity cover.bedroom_blinds state 'open' cannot be processed as a number
2023-12-12 17:28:10.677 WARNING (MainThread) [homeassistant.components.automation] Error evaluating condition in 'Bedroom - open window detection':
In 'condition':
  In 'state':
    In 'state' condition: unknown entity binary_sensor.openclose_13
2023-12-12 17:28:10.689 WARNING (MainThread) [homeassistant.components.homeassistant.triggers.numeric_state] Error in 'Living rooms - lights on when blinds close' trigger: In 'numeric_state' condition: entity cover.living_room_blinds state 'open' cannot be processed as a number
2023-12-12 17:28:11.120 WARNING (MainThread) [homeassistant.components.homeassistant.triggers.numeric_state] Error in 'Bedroom - lights off when blinds open' trigger: In 'numeric_state' condition: entity cover.bedroom_blinds state 'open' cannot be processed as a number
2023-12-12 17:28:11.123 WARNING (MainThread) [homeassistant.components.automation] Error evaluating condition in 'Bedroom - open window detection':
In 'condition':
  In 'state':
    In 'state' condition: unknown entity binary_sensor.openclose_13
2023-12-12 17:28:11.537 WARNING (MainThread) [homeassistant.components.homeassistant.triggers.numeric_state] Error in 'Bedroom - lights off when blinds open' trigger: In 'numeric_state' condition: entity cover.bedroom_blinds state 'open' cannot be processed as a number
2023-12-12 17:28:11.543 WARNING (MainThread) [homeassistant.components.automation] Error evaluating condition in 'Bedroom - open window detection':
In 'condition':
  In 'state':
    In 'state' condition: unknown entity binary_sensor.openclose_13
2023-12-12 17:28:11.604 WARNING (MainThread) [homeassistant.components.homeassistant.triggers.numeric_state] Error in 'Living rooms - lights on when blinds close' trigger: In 'numeric_state' condition: entity cover.living_room_blinds state 'open' cannot be processed as a number
2023-12-12 17:28:12.030 WARNING (MainThread) [homeassistant.components.homeassistant.triggers.numeric_state] Error in 'Bedroom - lights off when blinds open' trigger: In 'numeric_state' condition: entity cover.bedroom_blinds state 'open' cannot be processed as a number
2023-12-12 17:28:12.031 WARNING (MainThread) [homeassistant.components.automation] Error evaluating condition in 'Bedroom - open window detection':
In 'condition':
  In 'state':
    In 'state' condition: unknown entity binary_sensor.openclose_13
2023-12-12 17:28:12.479 WARNING (MainThread) [homeassistant.components.homeassistant.triggers.numeric_state] Error in 'Bedroom - lights off when blinds open' trigger: In 'numeric_state' condition: entity cover.bedroom_blinds state 'open' cannot be processed as a number
2023-12-12 17:28:12.482 WARNING (MainThread) [homeassistant.components.automation] Error evaluating condition in 'Bedroom - open window detection':
In 'condition':
  In 'state':
    In 'state' condition: unknown entity binary_sensor.openclose_13
2023-12-12 17:28:12.547 WARNING (MainThread) [homeassistant.components.homeassistant.triggers.numeric_state] Error in 'Living rooms - lights on when blinds close' trigger: In 'numeric_state' condition: entity cover.living_room_blinds state 'open' cannot be processed as a number
2023-12-12 17:28:12.861 WARNING (MainThread) [homeassistant.components.homeassistant.triggers.numeric_state] Error in 'Bedroom - lights off when blinds open' trigger: In 'numeric_state' condition: entity cover.bedroom_blinds state 'open' cannot be processed as a number
2023-12-12 17:28:12.863 WARNING (MainThread) [homeassistant.components.automation] Error evaluating condition in 'Bedroom - open window detection':
In 'condition':
  In 'state':
    In 'state' condition: unknown entity binary_sensor.openclose_13
2023-12-12 17:28:13.322 WARNING (MainThread) [homeassistant.components.homeassistant.triggers.numeric_state] Error in 'Bedroom - lights off when blinds open' trigger: In 'numeric_state' condition: entity cover.bedroom_blinds state 'open' cannot be processed as a number
2023-12-12 17:28:13.325 WARNING (MainThread) [homeassistant.components.automation] Error evaluating condition in 'Bedroom - open window detection':
In 'condition':
  In 'state':
    In 'state' condition: unknown entity binary_sensor.openclose_13
2023-12-12 17:28:13.532 WARNING (MainThread) [homeassistant.components.homeassistant.triggers.numeric_state] Error in 'Living rooms - lights on when blinds close' trigger: In 'numeric_state' condition: entity cover.living_room_blinds state 'open' cannot be processed as a number
2023-12-12 17:28:13.730 WARNING (MainThread) [homeassistant.components.homeassistant.triggers.numeric_state] Error in 'Bedroom - lights off when blinds open' trigger: In 'numeric_state' condition: entity cover.bedroom_blinds state 'open' cannot be processed as a number
2023-12-12 17:28:13.733 WARNING (MainThread) [homeassistant.components.automation] Error evaluating condition in 'Bedroom - open window detection':
In 'condition':
  In 'state':
    In 'state' condition: unknown entity binary_sensor.openclose_13
2023-12-12 17:28:14.307 WARNING (MainThread) [homeassistant.components.homeassistant.triggers.numeric_state] Error in 'Bedroom - lights off when blinds open' trigger: In 'numeric_state' condition: entity cover.bedroom_blinds state 'open' cannot be processed as a number
2023-12-12 17:28:14.311 WARNING (MainThread) [homeassistant.components.automation] Error evaluating condition in 'Bedroom - open window detection':
In 'condition':
  In 'state':
    In 'state' condition: unknown entity binary_sensor.openclose_13
2023-12-12 17:28:14.479 WARNING (MainThread) [homeassistant.components.homeassistant.triggers.numeric_state] Error in 'Living rooms - lights on when blinds close' trigger: In 'numeric_state' condition: entity cover.living_room_blinds state 'open' cannot be processed as a number
2023-12-12 17:28:14.658 WARNING (MainThread) [homeassistant.components.homeassistant.triggers.numeric_state] Error in 'Bedroom - lights off when blinds open' trigger: In 'numeric_state' condition: entity cover.bedroom_blinds state 'open' cannot be processed as a number
2023-12-12 17:28:14.660 WARNING (MainThread) [homeassistant.components.automation] Error evaluating condition in 'Bedroom - open window detection':
In 'condition':
  In 'state':
    In 'state' condition: unknown entity binary_sensor.openclose_13
2023-12-12 17:28:15.123 WARNING (MainThread) [homeassistant.components.homeassistant.triggers.numeric_state] Error in 'Bedroom - lights off when blinds open' trigger: In 'numeric_state' condition: entity cover.bedroom_blinds state 'open' cannot be processed as a number
2023-12-12 17:28:15.126 WARNING (MainThread) [homeassistant.components.automation] Error evaluating condition in 'Bedroom - open window detection':
In 'condition':
  In 'state':
    In 'state' condition: unknown entity binary_sensor.openclose_13
2023-12-12 17:28:15.386 WARNING (MainThread) [homeassistant.components.homeassistant.triggers.numeric_state] Error in 'Living rooms - lights on when blinds close' trigger: In 'numeric_state' condition: entity cover.living_room_blinds state 'open' cannot be processed as a number
2023-12-12 17:28:15.521 WARNING (MainThread) [homeassistant.components.homeassistant.triggers.numeric_state] Error in 'Bedroom - lights off when blinds open' trigger: In 'numeric_state' condition: entity cover.bedroom_blinds state 'open' cannot be processed as a number
2023-12-12 17:28:15.523 WARNING (MainThread) [homeassistant.components.automation] Error evaluating condition in 'Bedroom - open window detection':
In 'condition':
  In 'state':
    In 'state' condition: unknown entity binary_sensor.openclose_13
2023-12-12 17:28:15.982 WARNING (MainThread) [homeassistant.components.homeassistant.triggers.numeric_state] Error in 'Bedroom - lights off when blinds open' trigger: In 'numeric_state' condition: entity cover.bedroom_blinds state 'open' cannot be processed as a number
2023-12-12 17:28:15.985 WARNING (MainThread) [homeassistant.components.automation] Error evaluating condition in 'Bedroom - open window detection':
In 'condition':
  In 'state':
    In 'state' condition: unknown entity binary_sensor.openclose_13
2023-12-12 17:28:16.348 WARNING (MainThread) [homeassistant.components.homeassistant.triggers.numeric_state] Error in 'Living rooms - lights on when blinds close' trigger: In 'numeric_state' condition: entity cover.living_room_blinds state 'open' cannot be processed as a number
2023-12-12 17:28:16.444 WARNING (MainThread) [homeassistant.components.homeassistant.triggers.numeric_state] Error in 'Bedroom - lights off when blinds open' trigger: In 'numeric_state' condition: entity cover.bedroom_blinds state 'open' cannot be processed as a number
2023-12-12 17:28:16.446 WARNING (MainThread) [homeassistant.components.automation] Error evaluating condition in 'Bedroom - open window detection':
In 'condition':
  In 'state':
    In 'state' condition: unknown entity binary_sensor.openclose_13
2023-12-12 17:28:16.869 WARNING (MainThread) [homeassistant.components.homeassistant.triggers.numeric_state] Error in 'Bedroom - lights off when blinds open' trigger: In 'numeric_state' condition: entity cover.bedroom_blinds state 'open' cannot be processed as a number
2023-12-12 17:28:16.872 WARNING (MainThread) [homeassistant.components.automation] Error evaluating condition in 'Bedroom - open window detection':
In 'condition':
  In 'state':
    In 'state' condition: unknown entity binary_sensor.openclose_13
2023-12-12 17:28:17.318 WARNING (MainThread) [homeassistant.components.homeassistant.triggers.numeric_state] Error in 'Bedroom - lights off when blinds open' trigger: In 'numeric_state' condition: entity cover.bedroom_blinds state 'open' cannot be processed as a number
2023-12-12 17:28:17.322 WARNING (MainThread) [homeassistant.components.automation] Error evaluating condition in 'Bedroom - open window detection':
In 'condition':
  In 'state':
    In 'state' condition: unknown entity binary_sensor.openclose_13
2023-12-12 17:28:17.334 WARNING (MainThread) [homeassistant.components.homeassistant.triggers.numeric_state] Error in 'Living rooms - lights on when blinds close' trigger: In 'numeric_state' condition: entity cover.living_room_blinds state 'open' cannot be processed as a number
2023-12-12 17:28:17.773 WARNING (MainThread) [homeassistant.components.homeassistant.triggers.numeric_state] Error in 'Bedroom - lights off when blinds open' trigger: In 'numeric_state' condition: entity cover.bedroom_blinds state 'open' cannot be processed as a number
2023-12-12 17:28:17.778 WARNING (MainThread) [homeassistant.components.automation] Error evaluating condition in 'Bedroom - open window detection':
In 'condition':
  In 'state':
    In 'state' condition: unknown entity binary_sensor.openclose_13
2023-12-12 17:28:18.330 WARNING (MainThread) [homeassistant.components.homeassistant.triggers.numeric_state] Error in 'Living rooms - lights on when blinds close' trigger: In 'numeric_state' condition: entity cover.living_room_blinds state 'open' cannot be processed as a number
2023-12-12 17:28:19.175 WARNING (MainThread) [homeassistant.components.homeassistant.triggers.numeric_state] Error in 'Living rooms - lights on when blinds close' trigger: In 'numeric_state' condition: entity cover.living_room_blinds state 'open' cannot be processed as a number
2023-12-12 17:28:19.473 WARNING (MainThread) [homeassistant.components.homeassistant.triggers.numeric_state] Error in 'Bedroom - lights off when blinds open' trigger: In 'numeric_state' condition: entity cover.bedroom_blinds state 'open' cannot be processed as a number
2023-12-12 17:28:19.476 WARNING (MainThread) [homeassistant.components.automation] Error evaluating condition in 'Bedroom - open window detection':
In 'condition':
  In 'state':
    In 'state' condition: unknown entity binary_sensor.openclose_13
2023-12-12 17:28:19.528 WARNING (MainThread) [homeassistant.components.homeassistant.triggers.numeric_state] Error in 'Bedroom - lights off when blinds open' trigger: In 'numeric_state' condition: entity cover.bedroom_blinds state 'open' cannot be processed as a number
2023-12-12 17:28:19.531 WARNING (MainThread) [homeassistant.components.automation] Error evaluating condition in 'Bedroom - open window detection':
In 'condition':
  In 'state':
    In 'state' condition: unknown entity binary_sensor.openclose_13
2023-12-12 17:28:19.576 WARNING (MainThread) [homeassistant.components.homeassistant.triggers.numeric_state] Error in 'Bedroom - lights off when blinds open' trigger: In 'numeric_state' condition: entity cover.bedroom_blinds state 'open' cannot be processed as a number
2023-12-12 17:28:19.579 WARNING (MainThread) [homeassistant.components.automation] Error evaluating condition in 'Bedroom - open window detection':
In 'condition':
  In 'state':
    In 'state' condition: unknown entity binary_sensor.openclose_13
2023-12-12 17:28:19.710 WARNING (MainThread) [homeassistant.components.homeassistant.triggers.numeric_state] Error in 'Bedroom - lights off when blinds open' trigger: In 'numeric_state' condition: entity cover.bedroom_blinds state 'open' cannot be processed as a number
2023-12-12 17:28:19.713 WARNING (MainThread) [homeassistant.components.automation] Error evaluating condition in 'Bedroom - open window detection':
In 'condition':
  In 'state':
    In 'state' condition: unknown entity binary_sensor.openclose_13
2023-12-12 17:28:20.012 WARNING (MainThread) [homeassistant.components.homeassistant.triggers.numeric_state] Error in 'Bedroom - lights off when blinds open' trigger: In 'numeric_state' condition: entity cover.bedroom_blinds state 'open' cannot be processed as a number
2023-12-12 17:28:20.015 WARNING (MainThread) [homeassistant.components.automation] Error evaluating condition in 'Bedroom - open window detection':
In 'condition':
  In 'state':
    In 'state' condition: unknown entity binary_sensor.openclose_13
2023-12-12 17:28:20.186 WARNING (MainThread) [homeassistant.components.homeassistant.triggers.numeric_state] Error in 'Living rooms - lights on when blinds close' trigger: In 'numeric_state' condition: entity cover.living_room_blinds state 'open' cannot be processed as a number
2023-12-12 17:28:20.454 WARNING (MainThread) [homeassistant.components.homeassistant.triggers.numeric_state] Error in 'Bedroom - lights off when blinds open' trigger: In 'numeric_state' condition: entity cover.bedroom_blinds state 'open' cannot be processed as a number
2023-12-12 17:28:20.456 WARNING (MainThread) [homeassistant.components.automation] Error evaluating condition in 'Bedroom - open window detection':
In 'condition':
  In 'state':
    In 'state' condition: unknown entity binary_sensor.openclose_13
2023-12-12 17:28:20.874 WARNING (MainThread) [homeassistant.components.homeassistant.triggers.numeric_state] Error in 'Bedroom - lights off when blinds open' trigger: In 'numeric_state' condition: entity cover.bedroom_blinds state 'open' cannot be processed as a number
2023-12-12 17:28:20.876 WARNING (MainThread) [homeassistant.components.automation] Error evaluating condition in 'Bedroom - open window detection':
In 'condition':
  In 'state':
    In 'state' condition: unknown entity binary_sensor.openclose_13
2023-12-12 17:28:21.104 WARNING (MainThread) [homeassistant.components.homeassistant.triggers.numeric_state] Error in 'Living rooms - lights on when blinds close' trigger: In 'numeric_state' condition: entity cover.living_room_blinds state 'open' cannot be processed as a number
2023-12-12 17:28:21.343 WARNING (MainThread) [homeassistant.components.homeassistant.triggers.numeric_state] Error in 'Bedroom - lights off when blinds open' trigger: In 'numeric_state' condition: entity cover.bedroom_blinds state 'open' cannot be processed as a number
2023-12-12 17:28:21.347 WARNING (MainThread) [homeassistant.components.automation] Error evaluating condition in 'Bedroom - open window detection':
In 'condition':
  In 'state':
    In 'state' condition: unknown entity binary_sensor.openclose_13
2023-12-12 17:28:21.790 WARNING (MainThread) [homeassistant.components.homeassistant.triggers.numeric_state] Error in 'Bedroom - lights off when blinds open' trigger: In 'numeric_state' condition: entity cover.bedroom_blinds state 'open' cannot be processed as a number
2023-12-12 17:28:21.792 WARNING (MainThread) [homeassistant.components.automation] Error evaluating condition in 'Bedroom - open window detection':
In 'condition':
  In 'state':
    In 'state' condition: unknown entity binary_sensor.openclose_13
2023-12-12 17:28:22.035 WARNING (MainThread) [homeassistant.components.homeassistant.triggers.numeric_state] Error in 'Living rooms - lights on when blinds close' trigger: In 'numeric_state' condition: entity cover.living_room_blinds state 'open' cannot be processed as a number
2023-12-12 17:28:22.249 WARNING (MainThread) [homeassistant.components.homeassistant.triggers.numeric_state] Error in 'Bedroom - lights off when blinds open' trigger: In 'numeric_state' condition: entity cover.bedroom_blinds state 'open' cannot be processed as a number
2023-12-12 17:28:22.251 WARNING (MainThread) [homeassistant.components.automation] Error evaluating condition in 'Bedroom - open window detection':
In 'condition':
  In 'state':
    In 'state' condition: unknown entity binary_sensor.openclose_13
2023-12-12 17:28:22.670 WARNING (MainThread) [homeassistant.components.homeassistant.triggers.numeric_state] Error in 'Bedroom - lights off when blinds open' trigger: In 'numeric_state' condition: entity cover.bedroom_blinds state 'open' cannot be processed as a number
2023-12-12 17:28:22.673 WARNING (MainThread) [homeassistant.components.automation] Error evaluating condition in 'Bedroom - open window detection':
In 'condition':
  In 'state':
    In 'state' condition: unknown entity binary_sensor.openclose_13
2023-12-12 17:28:22.957 WARNING (MainThread) [homeassistant.components.homeassistant.triggers.numeric_state] Error in 'Living rooms - lights on when blinds close' trigger: In 'numeric_state' condition: entity cover.living_room_blinds state 'open' cannot be processed as a number
2023-12-12 17:28:23.129 WARNING (MainThread) [homeassistant.components.homeassistant.triggers.numeric_state] Error in 'Bedroom - lights off when blinds open' trigger: In 'numeric_state' condition: entity cover.bedroom_blinds state 'open' cannot be processed as a number
2023-12-12 17:28:23.131 WARNING (MainThread) [homeassistant.components.automation] Error evaluating condition in 'Bedroom - open window detection':
In 'condition':
  In 'state':
    In 'state' condition: unknown entity binary_sensor.openclose_13
2023-12-12 17:28:23.527 WARNING (MainThread) [homeassistant.components.homeassistant.triggers.numeric_state] Error in 'Bedroom - lights off when blinds open' trigger: In 'numeric_state' condition: entity cover.bedroom_blinds state 'open' cannot be processed as a number
2023-12-12 17:28:23.528 WARNING (MainThread) [homeassistant.components.automation] Error evaluating condition in 'Bedroom - open window detection':
In 'condition':
  In 'state':
    In 'state' condition: unknown entity binary_sensor.openclose_13
2023-12-12 17:28:23.931 WARNING (MainThread) [homeassistant.components.homeassistant.triggers.numeric_state] Error in 'Living rooms - lights on when blinds close' trigger: In 'numeric_state' condition: entity cover.living_room_blinds state 'open' cannot be processed as a number
2023-12-12 17:28:23.986 WARNING (MainThread) [homeassistant.components.homeassistant.triggers.numeric_state] Error in 'Bedroom - lights off when blinds open' trigger: In 'numeric_state' condition: entity cover.bedroom_blinds state 'open' cannot be processed as a number
2023-12-12 17:28:23.988 WARNING (MainThread) [homeassistant.components.automation] Error evaluating condition in 'Bedroom - open window detection':
In 'condition':
  In 'state':
    In 'state' condition: unknown entity binary_sensor.openclose_13
2023-12-12 17:28:24.447 WARNING (MainThread) [homeassistant.components.homeassistant.triggers.numeric_state] Error in 'Bedroom - lights off when blinds open' trigger: In 'numeric_state' condition: entity cover.bedroom_blinds state 'open' cannot be processed as a number
2023-12-12 17:28:24.448 WARNING (MainThread) [homeassistant.components.automation] Error evaluating condition in 'Bedroom - open window detection':
In 'condition':
  In 'state':
    In 'state' condition: unknown entity binary_sensor.openclose_13
2023-12-12 17:28:24.855 WARNING (MainThread) [homeassistant.components.homeassistant.triggers.numeric_state] Error in 'Bedroom - lights off when blinds open' trigger: In 'numeric_state' condition: entity cover.bedroom_blinds state 'open' cannot be processed as a number
2023-12-12 17:28:24.859 WARNING (MainThread) [homeassistant.components.automation] Error evaluating condition in 'Bedroom - open window detection':
In 'condition':
  In 'state':
    In 'state' condition: unknown entity binary_sensor.openclose_13
2023-12-12 17:28:24.873 WARNING (MainThread) [homeassistant.components.homeassistant.triggers.numeric_state] Error in 'Living rooms - lights on when blinds close' trigger: In 'numeric_state' condition: entity cover.living_room_blinds state 'open' cannot be processed as a number
2023-12-12 17:28:25.304 WARNING (MainThread) [homeassistant.components.homeassistant.triggers.numeric_state] Error in 'Bedroom - lights off when blinds open' trigger: In 'numeric_state' condition: entity cover.bedroom_blinds state 'open' cannot be processed as a number
2023-12-12 17:28:25.307 WARNING (MainThread) [homeassistant.components.automation] Error evaluating condition in 'Bedroom - open window detection':
In 'condition':
  In 'state':
    In 'state' condition: unknown entity binary_sensor.openclose_13
2023-12-12 17:28:25.775 WARNING (MainThread) [homeassistant.components.homeassistant.triggers.numeric_state] Error in 'Bedroom - lights off when blinds open' trigger: In 'numeric_state' condition: entity cover.bedroom_blinds state 'open' cannot be processed as a number
2023-12-12 17:28:25.777 WARNING (MainThread) [homeassistant.components.automation] Error evaluating condition in 'Bedroom - open window detection':
In 'condition':
  In 'state':
    In 'state' condition: unknown entity binary_sensor.openclose_13
2023-12-12 17:28:25.879 WARNING (MainThread) [homeassistant.components.homeassistant.triggers.numeric_state] Error in 'Living rooms - lights on when blinds close' trigger: In 'numeric_state' condition: entity cover.living_room_blinds state 'open' cannot be processed as a number
2023-12-12 17:28:26.237 WARNING (MainThread) [homeassistant.components.homeassistant.triggers.numeric_state] Error in 'Bedroom - lights off when blinds open' trigger: In 'numeric_state' condition: entity cover.bedroom_blinds state 'open' cannot be processed as a number
2023-12-12 17:28:26.240 WARNING (MainThread) [homeassistant.components.automation] Error evaluating condition in 'Bedroom - open window detection':
In 'condition':
  In 'state':
    In 'state' condition: unknown entity binary_sensor.openclose_13
2023-12-12 17:28:26.660 WARNING (MainThread) [homeassistant.components.homeassistant.triggers.numeric_state] Error in 'Bedroom - lights off when blinds open' trigger: In 'numeric_state' condition: entity cover.bedroom_blinds state 'open' cannot be processed as a number
2023-12-12 17:28:26.663 WARNING (MainThread) [homeassistant.components.automation] Error evaluating condition in 'Bedroom - open window detection':
In 'condition':
  In 'state':
    In 'state' condition: unknown entity binary_sensor.openclose_13
2023-12-12 17:28:26.749 WARNING (MainThread) [homeassistant.components.homeassistant.triggers.numeric_state] Error in 'Living rooms - lights on when blinds close' trigger: In 'numeric_state' condition: entity cover.living_room_blinds state 'open' cannot be processed as a number
2023-12-12 17:28:27.127 WARNING (MainThread) [homeassistant.components.homeassistant.triggers.numeric_state] Error in 'Bedroom - lights off when blinds open' trigger: In 'numeric_state' condition: entity cover.bedroom_blinds state 'open' cannot be processed as a number
2023-12-12 17:28:27.136 WARNING (MainThread) [homeassistant.components.automation] Error evaluating condition in 'Bedroom - open window detection':
In 'condition':
  In 'state':
    In 'state' condition: unknown entity binary_sensor.openclose_13
2023-12-12 17:28:27.558 WARNING (MainThread) [homeassistant.components.homeassistant.triggers.numeric_state] Error in 'Bedroom - lights off when blinds open' trigger: In 'numeric_state' condition: entity cover.bedroom_blinds state 'open' cannot be processed as a number
2023-12-12 17:28:27.561 WARNING (MainThread) [homeassistant.components.automation] Error evaluating condition in 'Bedroom - open window detection':
In 'condition':
  In 'state':
    In 'state' condition: unknown entity binary_sensor.openclose_13
2023-12-12 17:28:27.708 WARNING (MainThread) [homeassistant.components.homeassistant.triggers.numeric_state] Error in 'Living rooms - lights on when blinds close' trigger: In 'numeric_state' condition: entity cover.living_room_blinds state 'open' cannot be processed as a number
2023-12-12 17:28:27.972 WARNING (MainThread) [homeassistant.components.homeassistant.triggers.numeric_state] Error in 'Bedroom - lights off when blinds open' trigger: In 'numeric_state' condition: entity cover.bedroom_blinds state 'open' cannot be processed as a number
2023-12-12 17:28:27.974 WARNING (MainThread) [homeassistant.components.automation] Error evaluating condition in 'Bedroom - open window detection':
In 'condition':
  In 'state':
    In 'state' condition: unknown entity binary_sensor.openclose_13
2023-12-12 17:28:28.445 WARNING (MainThread) [homeassistant.components.homeassistant.triggers.numeric_state] Error in 'Bedroom - lights off when blinds open' trigger: In 'numeric_state' condition: entity cover.bedroom_blinds state 'open' cannot be processed as a number
2023-12-12 17:28:28.448 WARNING (MainThread) [homeassistant.components.automation] Error evaluating condition in 'Bedroom - open window detection':
In 'condition':
  In 'state':
    In 'state' condition: unknown entity binary_sensor.openclose_13
2023-12-12 17:28:28.668 WARNING (MainThread) [homeassistant.components.homeassistant.triggers.numeric_state] Error in 'Living rooms - lights on when blinds close' trigger: In 'numeric_state' condition: entity cover.living_room_blinds state 'open' cannot be processed as a number
2023-12-12 17:28:28.893 WARNING (MainThread) [homeassistant.components.homeassistant.triggers.numeric_state] Error in 'Bedroom - lights off when blinds open' trigger: In 'numeric_state' condition: entity cover.bedroom_blinds state 'open' cannot be processed as a number
2023-12-12 17:28:28.896 WARNING (MainThread) [homeassistant.components.automation] Error evaluating condition in 'Bedroom - open window detection':
In 'condition':
  In 'state':
    In 'state' condition: unknown entity binary_sensor.openclose_13
2023-12-12 17:28:29.356 WARNING (MainThread) [homeassistant.components.homeassistant.triggers.numeric_state] Error in 'Bedroom - lights off when blinds open' trigger: In 'numeric_state' condition: entity cover.bedroom_blinds state 'open' cannot be processed as a number
2023-12-12 17:28:29.359 WARNING (MainThread) [homeassistant.components.automation] Error evaluating condition in 'Bedroom - open window detection':
In 'condition':
  In 'state':
    In 'state' condition: unknown entity binary_sensor.openclose_13
2023-12-12 17:28:29.631 WARNING (MainThread) [homeassistant.components.homeassistant.triggers.numeric_state] Error in 'Living rooms - lights on when blinds close' trigger: In 'numeric_state' condition: entity cover.living_room_blinds state 'open' cannot be processed as a number
2023-12-12 17:28:29.766 WARNING (MainThread) [homeassistant.components.homeassistant.triggers.numeric_state] Error in 'Bedroom - lights off when blinds open' trigger: In 'numeric_state' condition: entity cover.bedroom_blinds state 'open' cannot be processed as a number
2023-12-12 17:28:29.770 WARNING (MainThread) [homeassistant.components.automation] Error evaluating condition in 'Bedroom - open window detection':
In 'condition':
  In 'state':
    In 'state' condition: unknown entity binary_sensor.openclose_13
2023-12-12 17:28:30.228 WARNING (MainThread) [homeassistant.components.homeassistant.triggers.numeric_state] Error in 'Bedroom - lights off when blinds open' trigger: In 'numeric_state' condition: entity cover.bedroom_blinds state 'open' cannot be processed as a number
2023-12-12 17:28:30.230 WARNING (MainThread) [homeassistant.components.automation] Error evaluating condition in 'Bedroom - open window detection':
In 'condition':
  In 'state':
    In 'state' condition: unknown entity binary_sensor.openclose_13
2023-12-12 17:28:30.553 WARNING (MainThread) [homeassistant.components.homeassistant.triggers.numeric_state] Error in 'Living rooms - lights on when blinds close' trigger: In 'numeric_state' condition: entity cover.living_room_blinds state 'open' cannot be processed as a number
2023-12-12 17:28:30.683 WARNING (MainThread) [homeassistant.components.homeassistant.triggers.numeric_state] Error in 'Bedroom - lights off when blinds open' trigger: In 'numeric_state' condition: entity cover.bedroom_blinds state 'open' cannot be processed as a number
2023-12-12 17:28:30.686 WARNING (MainThread) [homeassistant.components.automation] Error evaluating condition in 'Bedroom - open window detection':
In 'condition':
  In 'state':
    In 'state' condition: unknown entity binary_sensor.openclose_13
2023-12-12 17:28:31.099 WARNING (MainThread) [homeassistant.components.homeassistant.triggers.numeric_state] Error in 'Bedroom - lights off when blinds open' trigger: In 'numeric_state' condition: entity cover.bedroom_blinds state 'open' cannot be processed as a number
2023-12-12 17:28:31.105 WARNING (MainThread) [homeassistant.components.automation] Error evaluating condition in 'Bedroom - open window detection':
In 'condition':
  In 'state':
    In 'state' condition: unknown entity binary_sensor.openclose_13
2023-12-12 17:28:31.528 WARNING (MainThread) [homeassistant.components.homeassistant.triggers.numeric_state] Error in 'Living rooms - lights on when blinds close' trigger: In 'numeric_state' condition: entity cover.living_room_blinds state 'open' cannot be processed as a number
2023-12-12 17:28:31.557 WARNING (MainThread) [homeassistant.components.homeassistant.triggers.numeric_state] Error in 'Bedroom - lights off when blinds open' trigger: In 'numeric_state' condition: entity cover.bedroom_blinds state 'open' cannot be processed as a number
2023-12-12 17:28:31.560 WARNING (MainThread) [homeassistant.components.automation] Error evaluating condition in 'Bedroom - open window detection':
In 'condition':
  In 'state':
    In 'state' condition: unknown entity binary_sensor.openclose_13
2023-12-12 17:28:32.017 WARNING (MainThread) [homeassistant.components.homeassistant.triggers.numeric_state] Error in 'Bedroom - lights off when blinds open' trigger: In 'numeric_state' condition: entity cover.bedroom_blinds state 'open' cannot be processed as a number
2023-12-12 17:28:32.020 WARNING (MainThread) [homeassistant.components.automation] Error evaluating condition in 'Bedroom - open window detection':
In 'condition':
  In 'state':
    In 'state' condition: unknown entity binary_sensor.openclose_13
2023-12-12 17:28:32.423 WARNING (MainThread) [homeassistant.components.homeassistant.triggers.numeric_state] Error in 'Bedroom - lights off when blinds open' trigger: In 'numeric_state' condition: entity cover.bedroom_blinds state 'open' cannot be processed as a number
2023-12-12 17:28:32.425 WARNING (MainThread) [homeassistant.components.automation] Error evaluating condition in 'Bedroom - open window detection':
In 'condition':
  In 'state':
    In 'state' condition: unknown entity binary_sensor.openclose_13
2023-12-12 17:28:32.465 WARNING (MainThread) [homeassistant.components.homeassistant.triggers.numeric_state] Error in 'Living rooms - lights on when blinds close' trigger: In 'numeric_state' condition: entity cover.living_room_blinds state 'open' cannot be processed as a number
2023-12-12 17:28:32.881 WARNING (MainThread) [homeassistant.components.homeassistant.triggers.numeric_state] Error in 'Bedroom - lights off when blinds open' trigger: In 'numeric_state' condition: entity cover.bedroom_blinds state 'open' cannot be processed as a number
2023-12-12 17:28:32.884 WARNING (MainThread) [homeassistant.components.automation] Error evaluating condition in 'Bedroom - open window detection':
In 'condition':
  In 'state':
    In 'state' condition: unknown entity binary_sensor.openclose_13
2023-12-12 17:28:33.339 WARNING (MainThread) [homeassistant.components.homeassistant.triggers.numeric_state] Error in 'Bedroom - lights off when blinds open' trigger: In 'numeric_state' condition: entity cover.bedroom_blinds state 'open' cannot be processed as a number
2023-12-12 17:28:33.341 WARNING (MainThread) [homeassistant.components.automation] Error evaluating condition in 'Bedroom - open window detection':
In 'condition':
  In 'state':
    In 'state' condition: unknown entity binary_sensor.openclose_13
2023-12-12 17:28:33.405 WARNING (MainThread) [homeassistant.components.homeassistant.triggers.numeric_state] Error in 'Living rooms - lights on when blinds close' trigger: In 'numeric_state' condition: entity cover.living_room_blinds state 'open' cannot be processed as a number
2023-12-12 17:28:33.814 WARNING (MainThread) [homeassistant.components.homeassistant.triggers.numeric_state] Error in 'Bedroom - lights off when blinds open' trigger: In 'numeric_state' condition: entity cover.bedroom_blinds state 'open' cannot be processed as a number
2023-12-12 17:28:33.817 WARNING (MainThread) [homeassistant.components.automation] Error evaluating condition in 'Bedroom - open window detection':
In 'condition':
  In 'state':
    In 'state' condition: unknown entity binary_sensor.openclose_13
2023-12-12 17:28:34.218 WARNING (MainThread) [homeassistant.components.homeassistant.triggers.numeric_state] Error in 'Bedroom - lights off when blinds open' trigger: In 'numeric_state' condition: entity cover.bedroom_blinds state 'open' cannot be processed as a number
2023-12-12 17:28:34.220 WARNING (MainThread) [homeassistant.components.automation] Error evaluating condition in 'Bedroom - open window detection':
In 'condition':
  In 'state':
    In 'state' condition: unknown entity binary_sensor.openclose_13
2023-12-12 17:28:34.338 WARNING (MainThread) [homeassistant.components.homeassistant.triggers.numeric_state] Error in 'Living rooms - lights on when blinds close' trigger: In 'numeric_state' condition: entity cover.living_room_blinds state 'open' cannot be processed as a number
2023-12-12 17:28:34.700 WARNING (MainThread) [homeassistant.components.homeassistant.triggers.numeric_state] Error in 'Bedroom - lights off when blinds open' trigger: In 'numeric_state' condition: entity cover.bedroom_blinds state 'open' cannot be processed as a number
2023-12-12 17:28:34.702 WARNING (MainThread) [homeassistant.components.automation] Error evaluating condition in 'Bedroom - open window detection':
In 'condition':
  In 'state':
    In 'state' condition: unknown entity binary_sensor.openclose_13
2023-12-12 17:28:35.101 WARNING (MainThread) [homeassistant.components.homeassistant.triggers.numeric_state] Error in 'Bedroom - lights off when blinds open' trigger: In 'numeric_state' condition: entity cover.bedroom_blinds state 'open' cannot be processed as a number
2023-12-12 17:28:35.103 WARNING (MainThread) [homeassistant.components.automation] Error evaluating condition in 'Bedroom - open window detection':
In 'condition':
  In 'state':
    In 'state' condition: unknown entity binary_sensor.openclose_13
2023-12-12 17:28:35.319 WARNING (MainThread) [homeassistant.components.homeassistant.triggers.numeric_state] Error in 'Living rooms - lights on when blinds close' trigger: In 'numeric_state' condition: entity cover.living_room_blinds state 'open' cannot be processed as a number
2023-12-12 17:28:35.562 WARNING (MainThread) [homeassistant.components.homeassistant.triggers.numeric_state] Error in 'Bedroom - lights off when blinds open' trigger: In 'numeric_state' condition: entity cover.bedroom_blinds state 'open' cannot be processed as a number
2023-12-12 17:28:35.565 WARNING (MainThread) [homeassistant.components.automation] Error evaluating condition in 'Bedroom - open window detection':
In 'condition':
  In 'state':
    In 'state' condition: unknown entity binary_sensor.openclose_13
2023-12-12 17:28:36.075 WARNING (MainThread) [homeassistant.components.homeassistant.triggers.numeric_state] Error in 'Bedroom - lights off when blinds open' trigger: In 'numeric_state' condition: entity cover.bedroom_blinds state 'open' cannot be processed as a number
2023-12-12 17:28:36.078 WARNING (MainThread) [homeassistant.components.automation] Error evaluating condition in 'Bedroom - open window detection':
In 'condition':
  In 'state':
    In 'state' condition: unknown entity binary_sensor.openclose_13
2023-12-12 17:28:36.217 WARNING (MainThread) [homeassistant.components.homeassistant.triggers.numeric_state] Error in 'Living rooms - lights on when blinds close' trigger: In 'numeric_state' condition: entity cover.living_room_blinds state 'open' cannot be processed as a number
2023-12-12 17:28:36.423 WARNING (MainThread) [homeassistant.components.homeassistant.triggers.numeric_state] Error in 'Bedroom - lights off when blinds open' trigger: In 'numeric_state' condition: entity cover.bedroom_blinds state 'open' cannot be processed as a number
2023-12-12 17:28:36.426 WARNING (MainThread) [homeassistant.components.automation] Error evaluating condition in 'Bedroom - open window detection':
In 'condition':
  In 'state':
    In 'state' condition: unknown entity binary_sensor.openclose_13
2023-12-12 17:28:36.877 WARNING (MainThread) [homeassistant.components.homeassistant.triggers.numeric_state] Error in 'Bedroom - lights off when blinds open' trigger: In 'numeric_state' condition: entity cover.bedroom_blinds state 'open' cannot be processed as a number
2023-12-12 17:28:36.880 WARNING (MainThread) [homeassistant.components.automation] Error evaluating condition in 'Bedroom - open window detection':
In 'condition':
  In 'state':
    In 'state' condition: unknown entity binary_sensor.openclose_13
2023-12-12 17:28:37.189 WARNING (MainThread) [homeassistant.components.homeassistant.triggers.numeric_state] Error in 'Living rooms - lights on when blinds close' trigger: In 'numeric_state' condition: entity cover.living_room_blinds state 'open' cannot be processed as a number
2023-12-12 17:28:37.360 WARNING (MainThread) [homeassistant.components.homeassistant.triggers.numeric_state] Error in 'Bedroom - lights off when blinds open' trigger: In 'numeric_state' condition: entity cover.bedroom_blinds state 'open' cannot be processed as a number
2023-12-12 17:28:37.362 WARNING (MainThread) [homeassistant.components.automation] Error evaluating condition in 'Bedroom - open window detection':
In 'condition':
  In 'state':
    In 'state' condition: unknown entity binary_sensor.openclose_13
2023-12-12 17:28:37.863 WARNING (MainThread) [homeassistant.components.homeassistant.triggers.numeric_state] Error in 'Bedroom - lights off when blinds open' trigger: In 'numeric_state' condition: entity cover.bedroom_blinds state 'open' cannot be processed as a number
2023-12-12 17:28:37.866 WARNING (MainThread) [homeassistant.components.automation] Error evaluating condition in 'Bedroom - open window detection':
In 'condition':
  In 'state':
    In 'state' condition: unknown entity binary_sensor.openclose_13
2023-12-12 17:28:38.194 WARNING (MainThread) [homeassistant.components.homeassistant.triggers.numeric_state] Error in 'Living rooms - lights on when blinds close' trigger: In 'numeric_state' condition: entity cover.living_room_blinds state 'open' cannot be processed as a number
2023-12-12 17:28:38.211 WARNING (MainThread) [homeassistant.components.homeassistant.triggers.numeric_state] Error in 'Bedroom - lights off when blinds open' trigger: In 'numeric_state' condition: entity cover.bedroom_blinds state 'open' cannot be processed as a number
2023-12-12 17:28:38.212 WARNING (MainThread) [homeassistant.components.automation] Error evaluating condition in 'Bedroom - open window detection':
In 'condition':
  In 'state':
    In 'state' condition: unknown entity binary_sensor.openclose_13
2023-12-12 17:28:38.671 WARNING (MainThread) [homeassistant.components.homeassistant.triggers.numeric_state] Error in 'Bedroom - lights off when blinds open' trigger: In 'numeric_state' condition: entity cover.bedroom_blinds state 'open' cannot be processed as a number
2023-12-12 17:28:38.674 WARNING (MainThread) [homeassistant.components.automation] Error evaluating condition in 'Bedroom - open window detection':
In 'condition':
  In 'state':
    In 'state' condition: unknown entity binary_sensor.openclose_13
2023-12-12 17:28:39.060 WARNING (MainThread) [homeassistant.components.homeassistant.triggers.numeric_state] Error in 'Living rooms - lights on when blinds close' trigger: In 'numeric_state' condition: entity cover.living_room_blinds state 'open' cannot be processed as a number
2023-12-12 17:28:39.103 WARNING (MainThread) [homeassistant.components.homeassistant.triggers.numeric_state] Error in 'Bedroom - lights off when blinds open' trigger: In 'numeric_state' condition: entity cover.bedroom_blinds state 'open' cannot be processed as a number
2023-12-12 17:28:39.106 WARNING (MainThread) [homeassistant.components.automation] Error evaluating condition in 'Bedroom - open window detection':
In 'condition':
  In 'state':
    In 'state' condition: unknown entity binary_sensor.openclose_13
2023-12-12 17:28:39.539 WARNING (MainThread) [homeassistant.components.homeassistant.triggers.numeric_state] Error in 'Bedroom - lights off when blinds open' trigger: In 'numeric_state' condition: entity cover.bedroom_blinds state 'open' cannot be processed as a number
2023-12-12 17:28:39.542 WARNING (MainThread) [homeassistant.components.automation] Error evaluating condition in 'Bedroom - open window detection':
In 'condition':
  In 'state':
    In 'state' condition: unknown entity binary_sensor.openclose_13
2023-12-12 17:28:40.006 WARNING (MainThread) [homeassistant.components.homeassistant.triggers.numeric_state] Error in 'Bedroom - lights off when blinds open' trigger: In 'numeric_state' condition: entity cover.bedroom_blinds state 'open' cannot be processed as a number
2023-12-12 17:28:40.008 WARNING (MainThread) [homeassistant.components.automation] Error evaluating condition in 'Bedroom - open window detection':
In 'condition':
  In 'state':
    In 'state' condition: unknown entity binary_sensor.openclose_13
2023-12-12 17:28:40.018 WARNING (MainThread) [homeassistant.components.homeassistant.triggers.numeric_state] Error in 'Living rooms - lights on when blinds close' trigger: In 'numeric_state' condition: entity cover.living_room_blinds state 'open' cannot be processed as a number
2023-12-12 17:28:40.419 WARNING (MainThread) [homeassistant.components.homeassistant.triggers.numeric_state] Error in 'Bedroom - lights off when blinds open' trigger: In 'numeric_state' condition: entity cover.bedroom_blinds state 'open' cannot be processed as a number
2023-12-12 17:28:40.422 WARNING (MainThread) [homeassistant.components.automation] Error evaluating condition in 'Bedroom - open window detection':
In 'condition':
  In 'state':
    In 'state' condition: unknown entity binary_sensor.openclose_13
2023-12-12 17:28:40.868 WARNING (MainThread) [homeassistant.components.homeassistant.triggers.numeric_state] Error in 'Bedroom - lights off when blinds open' trigger: In 'numeric_state' condition: entity cover.bedroom_blinds state 'open' cannot be processed as a number
2023-12-12 17:28:40.872 WARNING (MainThread) [homeassistant.components.automation] Error evaluating condition in 'Bedroom - open window detection':
In 'condition':
  In 'state':
    In 'state' condition: unknown entity binary_sensor.openclose_13
2023-12-12 17:28:40.977 WARNING (MainThread) [homeassistant.components.homeassistant.triggers.numeric_state] Error in 'Living rooms - lights on when blinds close' trigger: In 'numeric_state' condition: entity cover.living_room_blinds state 'open' cannot be processed as a number
2023-12-12 17:28:41.343 WARNING (MainThread) [homeassistant.components.homeassistant.triggers.numeric_state] Error in 'Bedroom - lights off when blinds open' trigger: In 'numeric_state' condition: entity cover.bedroom_blinds state 'open' cannot be processed as a number
2023-12-12 17:28:41.346 WARNING (MainThread) [homeassistant.components.automation] Error evaluating condition in 'Bedroom - open window detection':
In 'condition':
  In 'state':
    In 'state' condition: unknown entity binary_sensor.openclose_13
2023-12-12 17:28:41.857 WARNING (MainThread) [homeassistant.components.homeassistant.triggers.numeric_state] Error in 'Bedroom - lights off when blinds open' trigger: In 'numeric_state' condition: entity cover.bedroom_blinds state 'open' cannot be processed as a number
2023-12-12 17:28:41.859 WARNING (MainThread) [homeassistant.components.automation] Error evaluating condition in 'Bedroom - open window detection':
In 'condition':
  In 'state':
    In 'state' condition: unknown entity binary_sensor.openclose_13
2023-12-12 17:28:41.931 WARNING (MainThread) [homeassistant.components.homeassistant.triggers.numeric_state] Error in 'Living rooms - lights on when blinds close' trigger: In 'numeric_state' condition: entity cover.living_room_blinds state 'open' cannot be processed as a number
2023-12-12 17:28:42.249 WARNING (MainThread) [homeassistant.components.homeassistant.triggers.numeric_state] Error in 'Bedroom - lights off when blinds open' trigger: In 'numeric_state' condition: entity cover.bedroom_blinds state 'open' cannot be processed as a number
2023-12-12 17:28:42.252 WARNING (MainThread) [homeassistant.components.automation] Error evaluating condition in 'Bedroom - open window detection':
In 'condition':
  In 'state':
    In 'state' condition: unknown entity binary_sensor.openclose_13
2023-12-12 17:28:42.705 WARNING (MainThread) [homeassistant.components.homeassistant.triggers.numeric_state] Error in 'Bedroom - lights off when blinds open' trigger: In 'numeric_state' condition: entity cover.bedroom_blinds state 'open' cannot be processed as a number
2023-12-12 17:28:42.708 WARNING (MainThread) [homeassistant.components.automation] Error evaluating condition in 'Bedroom - open window detection':
In 'condition':
  In 'state':
    In 'state' condition: unknown entity binary_sensor.openclose_13
2023-12-12 17:28:42.851 WARNING (MainThread) [homeassistant.components.homeassistant.triggers.numeric_state] Error in 'Living rooms - lights on when blinds close' trigger: In 'numeric_state' condition: entity cover.living_room_blinds state 'open' cannot be processed as a number
2023-12-12 17:28:43.133 WARNING (MainThread) [homeassistant.components.homeassistant.triggers.numeric_state] Error in 'Bedroom - lights off when blinds open' trigger: In 'numeric_state' condition: entity cover.bedroom_blinds state 'open' cannot be processed as a number
2023-12-12 17:28:43.135 WARNING (MainThread) [homeassistant.components.automation] Error evaluating condition in 'Bedroom - open window detection':
In 'condition':
  In 'state':
    In 'state' condition: unknown entity binary_sensor.openclose_13
2023-12-12 17:28:43.542 WARNING (MainThread) [homeassistant.components.homeassistant.triggers.numeric_state] Error in 'Bedroom - lights off when blinds open' trigger: In 'numeric_state' condition: entity cover.bedroom_blinds state 'open' cannot be processed as a number
2023-12-12 17:28:43.548 WARNING (MainThread) [homeassistant.components.automation] Error evaluating condition in 'Bedroom - open window detection':
In 'condition':
  In 'state':
    In 'state' condition: unknown entity binary_sensor.openclose_13
2023-12-12 17:28:43.799 WARNING (MainThread) [homeassistant.components.homeassistant.triggers.numeric_state] Error in 'Living rooms - lights on when blinds close' trigger: In 'numeric_state' condition: entity cover.living_room_blinds state 'open' cannot be processed as a number
2023-12-12 17:28:44.083 WARNING (MainThread) [homeassistant.components.homeassistant.triggers.numeric_state] Error in 'Bedroom - lights off when blinds open' trigger: In 'numeric_state' condition: entity cover.bedroom_blinds state 'open' cannot be processed as a number
2023-12-12 17:28:44.086 WARNING (MainThread) [homeassistant.components.automation] Error evaluating condition in 'Bedroom - open window detection':
In 'condition':
  In 'state':
    In 'state' condition: unknown entity binary_sensor.openclose_13
2023-12-12 17:28:44.530 WARNING (MainThread) [homeassistant.components.homeassistant.triggers.numeric_state] Error in 'Bedroom - lights off when blinds open' trigger: In 'numeric_state' condition: entity cover.bedroom_blinds state 'open' cannot be processed as a number
2023-12-12 17:28:44.533 WARNING (MainThread) [homeassistant.components.automation] Error evaluating condition in 'Bedroom - open window detection':
In 'condition':
  In 'state':
    In 'state' condition: unknown entity binary_sensor.openclose_13
2023-12-12 17:28:44.757 WARNING (MainThread) [homeassistant.components.homeassistant.triggers.numeric_state] Error in 'Living rooms - lights on when blinds close' trigger: In 'numeric_state' condition: entity cover.living_room_blinds state 'open' cannot be processed as a number
2023-12-12 17:28:44.905 WARNING (MainThread) [homeassistant.components.homeassistant.triggers.numeric_state] Error in 'Bedroom - lights off when blinds open' trigger: In 'numeric_state' condition: entity cover.bedroom_blinds state 'open' cannot be processed as a number
2023-12-12 17:28:44.907 WARNING (MainThread) [homeassistant.components.automation] Error evaluating condition in 'Bedroom - open window detection':
In 'condition':
  In 'state':
    In 'state' condition: unknown entity binary_sensor.openclose_13
2023-12-12 17:28:45.327 WARNING (MainThread) [homeassistant.components.homeassistant.triggers.numeric_state] Error in 'Bedroom - lights off when blinds open' trigger: In 'numeric_state' condition: entity cover.bedroom_blinds state 'open' cannot be processed as a number
2023-12-12 17:28:45.330 WARNING (MainThread) [homeassistant.components.automation] Error evaluating condition in 'Bedroom - open window detection':
In 'condition':
  In 'state':
    In 'state' condition: unknown entity binary_sensor.openclose_13
2023-12-12 17:28:45.682 WARNING (MainThread) [homeassistant.components.homeassistant.triggers.numeric_state] Error in 'Living rooms - lights on when blinds close' trigger: In 'numeric_state' condition: entity cover.living_room_blinds state 'open' cannot be processed as a number
2023-12-12 17:28:45.800 WARNING (MainThread) [homeassistant.components.homeassistant.triggers.numeric_state] Error in 'Bedroom - lights off when blinds open' trigger: In 'numeric_state' condition: entity cover.bedroom_blinds state 'open' cannot be processed as a number
2023-12-12 17:28:45.803 WARNING (MainThread) [homeassistant.components.automation] Error evaluating condition in 'Bedroom - open window detection':
In 'condition':
  In 'state':
    In 'state' condition: unknown entity binary_sensor.openclose_13
2023-12-12 17:28:46.212 WARNING (MainThread) [homeassistant.components.homeassistant.triggers.numeric_state] Error in 'Bedroom - lights off when blinds open' trigger: In 'numeric_state' condition: entity cover.bedroom_blinds state 'open' cannot be processed as a number
2023-12-12 17:28:46.215 WARNING (MainThread) [homeassistant.components.automation] Error evaluating condition in 'Bedroom - open window detection':
In 'condition':
  In 'state':
    In 'state' condition: unknown entity binary_sensor.openclose_13
2023-12-12 17:28:46.625 WARNING (MainThread) [homeassistant.components.homeassistant.triggers.numeric_state] Error in 'Living rooms - lights on when blinds close' trigger: In 'numeric_state' condition: entity cover.living_room_blinds state 'open' cannot be processed as a number
2023-12-12 17:28:46.656 WARNING (MainThread) [homeassistant.components.homeassistant.triggers.numeric_state] Error in 'Bedroom - lights off when blinds open' trigger: In 'numeric_state' condition: entity cover.bedroom_blinds state 'open' cannot be processed as a number
2023-12-12 17:28:46.658 WARNING (MainThread) [homeassistant.components.automation] Error evaluating condition in 'Bedroom - open window detection':
In 'condition':
  In 'state':
    In 'state' condition: unknown entity binary_sensor.openclose_13
2023-12-12 17:28:47.113 WARNING (MainThread) [homeassistant.components.homeassistant.triggers.numeric_state] Error in 'Bedroom - lights off when blinds open' trigger: In 'numeric_state' condition: entity cover.bedroom_blinds state 'open' cannot be processed as a number
2023-12-12 17:28:47.116 WARNING (MainThread) [homeassistant.components.automation] Error evaluating condition in 'Bedroom - open window detection':
In 'condition':
  In 'state':
    In 'state' condition: unknown entity binary_sensor.openclose_13
2023-12-12 17:28:47.541 WARNING (MainThread) [homeassistant.components.homeassistant.triggers.numeric_state] Error in 'Bedroom - lights off when blinds open' trigger: In 'numeric_state' condition: entity cover.bedroom_blinds state 'open' cannot be processed as a number
2023-12-12 17:28:47.544 WARNING (MainThread) [homeassistant.components.automation] Error evaluating condition in 'Bedroom - open window detection':
In 'condition':
  In 'state':
    In 'state' condition: unknown entity binary_sensor.openclose_13
2023-12-12 17:28:47.602 WARNING (MainThread) [homeassistant.components.homeassistant.triggers.numeric_state] Error in 'Living rooms - lights on when blinds close' trigger: In 'numeric_state' condition: entity cover.living_room_blinds state 'open' cannot be processed as a number
2023-12-12 17:28:47.979 WARNING (MainThread) [homeassistant.components.homeassistant.triggers.numeric_state] Error in 'Bedroom - lights off when blinds open' trigger: In 'numeric_state' condition: entity cover.bedroom_blinds state 'open' cannot be processed as a number
2023-12-12 17:28:47.983 WARNING (MainThread) [homeassistant.components.automation] Error evaluating condition in 'Bedroom - open window detection':
In 'condition':
  In 'state':
    In 'state' condition: unknown entity binary_sensor.openclose_13
2023-12-12 17:28:48.444 WARNING (MainThread) [homeassistant.components.homeassistant.triggers.numeric_state] Error in 'Bedroom - lights off when blinds open' trigger: In 'numeric_state' condition: entity cover.bedroom_blinds state 'open' cannot be processed as a number
2023-12-12 17:28:48.446 WARNING (MainThread) [homeassistant.components.automation] Error evaluating condition in 'Bedroom - open window detection':
In 'condition':
  In 'state':
    In 'state' condition: unknown entity binary_sensor.openclose_13
2023-12-12 17:28:48.610 WARNING (MainThread) [homeassistant.components.homeassistant.triggers.numeric_state] Error in 'Living rooms - lights on when blinds close' trigger: In 'numeric_state' condition: entity cover.living_room_blinds state 'open' cannot be processed as a number
2023-12-12 17:28:48.953 WARNING (MainThread) [homeassistant.components.homeassistant.triggers.numeric_state] Error in 'Bedroom - lights off when blinds open' trigger: In 'numeric_state' condition: entity cover.bedroom_blinds state 'open' cannot be processed as a number
2023-12-12 17:28:48.956 WARNING (MainThread) [homeassistant.components.automation] Error evaluating condition in 'Bedroom - open window detection':
In 'condition':
  In 'state':
    In 'state' condition: unknown entity binary_sensor.openclose_13
2023-12-12 17:28:49.318 WARNING (MainThread) [homeassistant.components.homeassistant.triggers.numeric_state] Error in 'Bedroom - lights off when blinds open' trigger: In 'numeric_state' condition: entity cover.bedroom_blinds state 'open' cannot be processed as a number
2023-12-12 17:28:49.323 WARNING (MainThread) [homeassistant.components.automation] Error evaluating condition in 'Bedroom - open window detection':
In 'condition':
  In 'state':
    In 'state' condition: unknown entity binary_sensor.openclose_13
2023-12-12 17:28:49.554 WARNING (MainThread) [homeassistant.components.homeassistant.triggers.numeric_state] Error in 'Living rooms - lights on when blinds close' trigger: In 'numeric_state' condition: entity cover.living_room_blinds state 'open' cannot be processed as a number
2023-12-12 17:28:49.775 WARNING (MainThread) [homeassistant.components.homeassistant.triggers.numeric_state] Error in 'Bedroom - lights off when blinds open' trigger: In 'numeric_state' condition: entity cover.bedroom_blinds state 'open' cannot be processed as a number
2023-12-12 17:28:49.778 WARNING (MainThread) [homeassistant.components.automation] Error evaluating condition in 'Bedroom - open window detection':
In 'condition':
  In 'state':
    In 'state' condition: unknown entity binary_sensor.openclose_13
2023-12-12 17:28:50.232 WARNING (MainThread) [homeassistant.components.homeassistant.triggers.numeric_state] Error in 'Bedroom - lights off when blinds open' trigger: In 'numeric_state' condition: entity cover.bedroom_blinds state 'open' cannot be processed as a number
2023-12-12 17:28:50.235 WARNING (MainThread) [homeassistant.components.automation] Error evaluating condition in 'Bedroom - open window detection':
In 'condition':
  In 'state':
    In 'state' condition: unknown entity binary_sensor.openclose_13
2023-12-12 17:28:50.423 WARNING (MainThread) [homeassistant.components.homeassistant.triggers.numeric_state] Error in 'Living rooms - lights on when blinds close' trigger: In 'numeric_state' condition: entity cover.living_room_blinds state 'open' cannot be processed as a number
2023-12-12 17:28:50.698 WARNING (MainThread) [homeassistant.components.homeassistant.triggers.numeric_state] Error in 'Bedroom - lights off when blinds open' trigger: In 'numeric_state' condition: entity cover.bedroom_blinds state 'open' cannot be processed as a number
2023-12-12 17:28:50.705 WARNING (MainThread) [homeassistant.components.automation] Error evaluating condition in 'Bedroom - open window detection':
In 'condition':
  In 'state':
    In 'state' condition: unknown entity binary_sensor.openclose_13
2023-12-12 17:28:51.110 WARNING (MainThread) [homeassistant.components.homeassistant.triggers.numeric_state] Error in 'Bedroom - lights off when blinds open' trigger: In 'numeric_state' condition: entity cover.bedroom_blinds state 'open' cannot be processed as a number
2023-12-12 17:28:51.113 WARNING (MainThread) [homeassistant.components.automation] Error evaluating condition in 'Bedroom - open window detection':
In 'condition':
  In 'state':
    In 'state' condition: unknown entity binary_sensor.openclose_13
2023-12-12 17:28:51.400 WARNING (MainThread) [homeassistant.components.homeassistant.triggers.numeric_state] Error in 'Living rooms - lights on when blinds close' trigger: In 'numeric_state' condition: entity cover.living_room_blinds state 'open' cannot be processed as a number
2023-12-12 17:28:51.566 WARNING (MainThread) [homeassistant.components.homeassistant.triggers.numeric_state] Error in 'Bedroom - lights off when blinds open' trigger: In 'numeric_state' condition: entity cover.bedroom_blinds state 'open' cannot be processed as a number
2023-12-12 17:28:51.569 WARNING (MainThread) [homeassistant.components.automation] Error evaluating condition in 'Bedroom - open window detection':
In 'condition':
  In 'state':
    In 'state' condition: unknown entity binary_sensor.openclose_13
2023-12-12 17:28:52.071 WARNING (MainThread) [homeassistant.components.homeassistant.triggers.numeric_state] Error in 'Bedroom - lights off when blinds open' trigger: In 'numeric_state' condition: entity cover.bedroom_blinds state 'closed' cannot be processed as a number
2023-12-12 17:28:52.074 WARNING (MainThread) [homeassistant.components.automation] Error evaluating condition in 'Bedroom - open window detection':
In 'condition':
  In 'state':
    In 'state' condition: unknown entity binary_sensor.openclose_13
2023-12-12 17:28:52.340 WARNING (MainThread) [homeassistant.components.homeassistant.triggers.numeric_state] Error in 'Living rooms - lights on when blinds close' trigger: In 'numeric_state' condition: entity cover.living_room_blinds state 'open' cannot be processed as a number
2023-12-12 17:28:53.266 WARNING (MainThread) [homeassistant.components.homeassistant.triggers.numeric_state] Error in 'Living rooms - lights on when blinds close' trigger: In 'numeric_state' condition: entity cover.living_room_blinds state 'open' cannot be processed as a number
2023-12-12 17:28:54.339 WARNING (MainThread) [homeassistant.components.homeassistant.triggers.numeric_state] Error in 'Living rooms - lights on when blinds close' trigger: In 'numeric_state' condition: entity cover.living_room_blinds state 'open' cannot be processed as a number
2023-12-12 17:28:55.170 WARNING (MainThread) [homeassistant.components.homeassistant.triggers.numeric_state] Error in 'Living rooms - lights on when blinds close' trigger: In 'numeric_state' condition: entity cover.living_room_blinds state 'open' cannot be processed as a number
2023-12-12 17:28:56.128 WARNING (MainThread) [homeassistant.components.homeassistant.triggers.numeric_state] Error in 'Living rooms - lights on when blinds close' trigger: In 'numeric_state' condition: entity cover.living_room_blinds state 'open' cannot be processed as a number
2023-12-12 17:28:57.057 WARNING (MainThread) [homeassistant.components.homeassistant.triggers.numeric_state] Error in 'Living rooms - lights on when blinds close' trigger: In 'numeric_state' condition: entity cover.living_room_blinds state 'open' cannot be processed as a number
2023-12-12 17:28:57.997 WARNING (MainThread) [homeassistant.components.homeassistant.triggers.numeric_state] Error in 'Living rooms - lights on when blinds close' trigger: In 'numeric_state' condition: entity cover.living_room_blinds state 'open' cannot be processed as a number
2023-12-12 17:28:58.958 WARNING (MainThread) [homeassistant.components.homeassistant.triggers.numeric_state] Error in 'Living rooms - lights on when blinds close' trigger: In 'numeric_state' condition: entity cover.living_room_blinds state 'open' cannot be processed as a number
2023-12-12 17:28:59.982 WARNING (MainThread) [homeassistant.components.homeassistant.triggers.numeric_state] Error in 'Living rooms - lights on when blinds close' trigger: In 'numeric_state' condition: entity cover.living_room_blinds state 'open' cannot be processed as a number
2023-12-12 17:29:00.950 WARNING (MainThread) [homeassistant.components.homeassistant.triggers.numeric_state] Error in 'Living rooms - lights on when blinds close' trigger: In 'numeric_state' condition: entity cover.living_room_blinds state 'open' cannot be processed as a number
2023-12-12 17:29:01.877 WARNING (MainThread) [homeassistant.components.homeassistant.triggers.numeric_state] Error in 'Living rooms - lights on when blinds close' trigger: In 'numeric_state' condition: entity cover.living_room_blinds state 'open' cannot be processed as a number
2023-12-12 17:29:02.755 WARNING (MainThread) [homeassistant.components.homeassistant.triggers.numeric_state] Error in 'Living rooms - lights on when blinds close' trigger: In 'numeric_state' condition: entity cover.living_room_blinds state 'open' cannot be processed as a number
2023-12-12 17:29:03.709 WARNING (MainThread) [homeassistant.components.homeassistant.triggers.numeric_state] Error in 'Living rooms - lights on when blinds close' trigger: In 'numeric_state' condition: entity cover.living_room_blinds state 'closed' cannot be processed as a number
2023-12-12 17:29:29.702 ERROR (SyncWorker_0) [homeassistant.components.command_line.utils] Command failed (with return code 1): python3 -c "import requests; import json; import random; dataRequest = requests.get('https://gpsgadget.buienradar.nl/data/raintext?lat=51.83&lon=5.86&c='+str(random.randint(0,999999999999999)) ).text; dataRequest = dataRequest.replace('\r\n',' '); data = '{\"data\":\"'+dataRequest+'\"}';    pri(data);"
2023-12-12 17:29:29.704 WARNING (MainThread) [homeassistant.components.command_line] Empty reply found when expecting JSON data
2023-12-12 17:33:29.700 ERROR (SyncWorker_1) [homeassistant.components.command_line.utils] Command failed (with return code 1): python3 -c "import requests; import json; import random; dataRequest = requests.get('https://gpsgadget.buienradar.nl/data/raintext?lat=51.83&lon=5.86&c='+str(random.randint(0,999999999999999)) ).text; dataRequest = dataRequest.replace('\r\n',' '); data = '{\"data\":\"'+dataRequest+'\"}';    pri(data);"
2023-12-12 17:33:29.702 WARNING (MainThread) [homeassistant.components.command_line] Empty reply found when expecting JSON data
2023-12-12 17:37:30.945 ERROR (SyncWorker_6) [homeassistant.components.command_line.utils] Command failed (with return code 1): python3 -c "import requests; import json; import random; dataRequest = requests.get('https://gpsgadget.buienradar.nl/data/raintext?lat=51.83&lon=5.86&c='+str(random.randint(0,999999999999999)) ).text; dataRequest = dataRequest.replace('\r\n',' '); data = '{\"data\":\"'+dataRequest+'\"}';    pri(data);"
2023-12-12 17:37:30.946 WARNING (MainThread) [homeassistant.components.command_line] Empty reply found when expecting JSON data
2023-12-12 17:41:30.918 ERROR (SyncWorker_9) [homeassistant.components.command_line.utils] Command failed (with return code 1): python3 -c "import requests; import json; import random; dataRequest = requests.get('https://gpsgadget.buienradar.nl/data/raintext?lat=51.83&lon=5.86&c='+str(random.randint(0,999999999999999)) ).text; dataRequest = dataRequest.replace('\r\n',' '); data = '{\"data\":\"'+dataRequest+'\"}';    pri(data);"
2023-12-12 17:41:30.920 WARNING (MainThread) [homeassistant.components.command_line] Empty reply found when expecting JSON data
2023-12-12 17:42:14.119 WARNING (MainThread) [homeassistant.helpers.template] Template variable warning: 'title' is undefined when rendering '{{ title }}'
2023-12-12 17:42:14.554 WARNING (MainThread) [homeassistant.helpers.template] Template variable warning: 'title' is undefined when rendering '{{ title }}'
2023-12-12 17:42:14.556 ERROR (MainThread) [homeassistant.components.script.notify_people_who_are_present] Notify people who are home: If at step 2: Error executing script. Service not found for call_service at pos 1: Service notify.mobile_app_user2_phone not found.
2023-12-12 17:42:14.557 ERROR (MainThread) [homeassistant.components.script.notify_people_who_are_present] Notify people who are home: Error executing script. Service not found for if at pos 2: Service notify.mobile_app_user2_phone not found.
2023-12-12 17:42:14.564 ERROR (MainThread) [homeassistant.components.automation.all_rooms_rain_alert_when_window_open] All rooms - Rain alert when window open: Error executing script. Service not found for call_service at pos 8: Service notify.mobile_app_user2_phone not found.
2023-12-12 17:45:30.899 ERROR (SyncWorker_3) [homeassistant.components.command_line.utils] Command failed (with return code 1): python3 -c "import requests; import json; import random; dataRequest = requests.get('https://gpsgadget.buienradar.nl/data/raintext?lat=51.83&lon=5.86&c='+str(random.randint(0,999999999999999)) ).text; dataRequest = dataRequest.replace('\r\n',' '); data = '{\"data\":\"'+dataRequest+'\"}';    pri(data);"
2023-12-12 17:45:30.901 WARNING (MainThread) [homeassistant.components.command_line] Empty reply found when expecting JSON data
2023-12-12 17:49:31.080 ERROR (SyncWorker_5) [homeassistant.components.command_line.utils] Command failed (with return code 1): python3 -c "import requests; import json; import random; dataRequest = requests.get('https://gpsgadget.buienradar.nl/data/raintext?lat=51.83&lon=5.86&c='+str(random.randint(0,999999999999999)) ).text; dataRequest = dataRequest.replace('\r\n',' '); data = '{\"data\":\"'+dataRequest+'\"}';    pri(data);"
2023-12-12 17:49:31.082 WARNING (MainThread) [homeassistant.components.command_line] Empty reply found when expecting JSON data
2023-12-12 17:53:30.859 ERROR (SyncWorker_6) [homeassistant.components.command_line.utils] Command failed (with return code 1): python3 -c "import requests; import json; import random; dataRequest = requests.get('https://gpsgadget.buienradar.nl/data/raintext?lat=51.83&lon=5.86&c='+str(random.randint(0,999999999999999)) ).text; dataRequest = dataRequest.replace('\r\n',' '); data = '{\"data\":\"'+dataRequest+'\"}';    pri(data);"
2023-12-12 17:53:30.861 WARNING (MainThread) [homeassistant.components.command_line] Empty reply found when expecting JSON data
2023-12-12 17:57:30.963 ERROR (SyncWorker_7) [homeassistant.components.command_line.utils] Command failed (with return code 1): python3 -c "import requests; import json; import random; dataRequest = requests.get('https://gpsgadget.buienradar.nl/data/raintext?lat=51.83&lon=5.86&c='+str(random.randint(0,999999999999999)) ).text; dataRequest = dataRequest.replace('\r\n',' '); data = '{\"data\":\"'+dataRequest+'\"}';    pri(data);"
2023-12-12 17:57:30.964 WARNING (MainThread) [homeassistant.components.command_line] Empty reply found when expecting JSON data
2023-12-12 18:01:30.941 ERROR (SyncWorker_6) [homeassistant.components.command_line.utils] Command failed (with return code 1): python3 -c "import requests; import json; import random; dataRequest = requests.get('https://gpsgadget.buienradar.nl/data/raintext?lat=51.83&lon=5.86&c='+str(random.randint(0,999999999999999)) ).text; dataRequest = dataRequest.replace('\r\n',' '); data = '{\"data\":\"'+dataRequest+'\"}';    pri(data);"
2023-12-12 18:01:30.943 WARNING (MainThread) [homeassistant.components.command_line] Empty reply found when expecting JSON data
2023-12-12 18:03:16.823 WARNING (MainThread) [homeassistant.helpers.script] Error in 'if[0]' evaluation: In 'numeric_state':
  In 'numeric_state' condition: unknown entity sensor.humidity_11
2023-12-12 18:05:30.950 ERROR (SyncWorker_6) [homeassistant.components.command_line.utils] Command failed (with return code 1): python3 -c "import requests; import json; import random; dataRequest = requests.get('https://gpsgadget.buienradar.nl/data/raintext?lat=51.83&lon=5.86&c='+str(random.randint(0,999999999999999)) ).text; dataRequest = dataRequest.replace('\r\n',' '); data = '{\"data\":\"'+dataRequest+'\"}';    pri(data);"
2023-12-12 18:05:30.952 WARNING (MainThread) [homeassistant.components.command_line] Empty reply found when expecting JSON data
2023-12-12 18:08:19.314 WARNING (MainThread) [homeassistant.helpers.template] Template variable warning: 'title' is undefined when rendering '{{ title }}'
2023-12-12 18:08:19.648 WARNING (MainThread) [homeassistant.helpers.template] Template variable warning: 'title' is undefined when rendering '{{ title }}'
2023-12-12 18:08:19.650 ERROR (MainThread) [homeassistant.components.script.notify_people_who_are_present] Notify people who are home: If at step 2: Error executing script. Service not found for call_service at pos 1: Service notify.mobile_app_user2_phone not found.
2023-12-12 18:08:19.652 ERROR (MainThread) [homeassistant.components.script.notify_people_who_are_present] Notify people who are home: Error executing script. Service not found for if at pos 2: Service notify.mobile_app_user2_phone not found.
2023-12-12 18:08:19.657 ERROR (MainThread) [homeassistant.components.automation.all_rooms_rain_alert_when_window_open] All rooms - Rain alert when window open: If at step 4: Error executing script. Service not found for call_service at pos 1: Service notify.mobile_app_user2_phone not found.
2023-12-12 18:08:19.658 ERROR (MainThread) [homeassistant.components.automation.all_rooms_rain_alert_when_window_open] All rooms - Rain alert when window open: Error executing script. Service not found for if at pos 4: Service notify.mobile_app_user2_phone not found.
2023-12-12 18:09:30.919 ERROR (SyncWorker_9) [homeassistant.components.command_line.utils] Command failed (with return code 1): python3 -c "import requests; import json; import random; dataRequest = requests.get('https://gpsgadget.buienradar.nl/data/raintext?lat=51.83&lon=5.86&c='+str(random.randint(0,999999999999999)) ).text; dataRequest = dataRequest.replace('\r\n',' '); data = '{\"data\":\"'+dataRequest+'\"}';    pri(data);"
2023-12-12 18:09:30.920 WARNING (MainThread) [homeassistant.components.command_line] Empty reply found when expecting JSON data
2023-12-12 18:12:14.129 WARNING (SyncWorker_2) [buienalarm.pybuienalarm] Error fetching data safely: 503 Server Error: Service Unavailable for url: https://cdn-secure.buienalarm.nl/api/3.4/forecast.php?lat=51.82973871337543&lon=5.858429971873754&region=nl&unit=mm%2Fu
2023-12-12 18:13:30.932 ERROR (SyncWorker_4) [homeassistant.components.command_line.utils] Command failed (with return code 1): python3 -c "import requests; import json; import random; dataRequest = requests.get('https://gpsgadget.buienradar.nl/data/raintext?lat=51.83&lon=5.86&c='+str(random.randint(0,999999999999999)) ).text; dataRequest = dataRequest.replace('\r\n',' '); data = '{\"data\":\"'+dataRequest+'\"}';    pri(data);"
2023-12-12 18:13:30.933 WARNING (MainThread) [homeassistant.components.command_line] Empty reply found when expecting JSON data
2023-12-12 18:17:30.941 ERROR (SyncWorker_5) [homeassistant.components.command_line.utils] Command failed (with return code 1): python3 -c "import requests; import json; import random; dataRequest = requests.get('https://gpsgadget.buienradar.nl/data/raintext?lat=51.83&lon=5.86&c='+str(random.randint(0,999999999999999)) ).text; dataRequest = dataRequest.replace('\r\n',' '); data = '{\"data\":\"'+dataRequest+'\"}';    pri(data);"
2023-12-12 18:17:30.943 WARNING (MainThread) [homeassistant.components.command_line] Empty reply found when expecting JSON data
2023-12-12 18:21:30.446 ERROR (MainThread) [homeassistant.components.automation.living_room_alert_if_litterbox_need_changing] Living room - Alert if litterbox need changing: Reset if cleaned: Error executing script. Service not found for call_service at pos 2: Service notify.mobile_app_user2_phone not found.
2023-12-12 18:21:30.447 ERROR (MainThread) [homeassistant.components.automation.living_room_alert_if_litterbox_need_changing] Living room - Alert if litterbox need changing: Error executing script. Service not found for if at pos 5: Service notify.mobile_app_user2_phone not found.
2023-12-12 18:21:30.937 ERROR (SyncWorker_2) [homeassistant.components.command_line.utils] Command failed (with return code 1): python3 -c "import requests; import json; import random; dataRequest = requests.get('https://gpsgadget.buienradar.nl/data/raintext?lat=51.83&lon=5.86&c='+str(random.randint(0,999999999999999)) ).text; dataRequest = dataRequest.replace('\r\n',' '); data = '{\"data\":\"'+dataRequest+'\"}';    pri(data);"
2023-12-12 18:21:30.939 WARNING (MainThread) [homeassistant.components.command_line] Empty reply found when expecting JSON data
2023-12-12 18:23:50.726 WARNING (MainThread) [homeassistant.helpers.template] Template variable warning: 'title' is undefined when rendering '{{ title }}'
2023-12-12 18:23:51.018 WARNING (MainThread) [homeassistant.helpers.template] Template variable warning: 'title' is undefined when rendering '{{ title }}'
2023-12-12 18:23:51.021 ERROR (MainThread) [homeassistant.components.script.notify_people_who_are_present] Notify people who are home: If at step 2: Error executing script. Service not found for call_service at pos 1: Service notify.mobile_app_user2_phone not found.
2023-12-12 18:23:51.022 ERROR (MainThread) [homeassistant.components.script.notify_people_who_are_present] Notify people who are home: Error executing script. Service not found for if at pos 2: Service notify.mobile_app_user2_phone not found.
2023-12-12 18:23:51.024 ERROR (MainThread) [homeassistant.components.automation.all_rooms_rain_alert_when_window_open] All rooms - Rain alert when window open: If at step 4: Error executing script. Service not found for call_service at pos 1: Service notify.mobile_app_user2_phone not found.
2023-12-12 18:23:51.025 ERROR (MainThread) [homeassistant.components.automation.all_rooms_rain_alert_when_window_open] All rooms - Rain alert when window open: Error executing script. Service not found for if at pos 4: Service notify.mobile_app_user2_phone not found.
2023-12-12 18:25:30.936 ERROR (SyncWorker_8) [homeassistant.components.command_line.utils] Command failed (with return code 1): python3 -c "import requests; import json; import random; dataRequest = requests.get('https://gpsgadget.buienradar.nl/data/raintext?lat=51.83&lon=5.86&c='+str(random.randint(0,999999999999999)) ).text; dataRequest = dataRequest.replace('\r\n',' '); data = '{\"data\":\"'+dataRequest+'\"}';    pri(data);"
2023-12-12 18:25:30.938 WARNING (MainThread) [homeassistant.components.command_line] Empty reply found when expecting JSON data
2023-12-12 18:29:30.905 ERROR (SyncWorker_5) [homeassistant.components.command_line.utils] Command failed (with return code 1): python3 -c "import requests; import json; import random; dataRequest = requests.get('https://gpsgadget.buienradar.nl/data/raintext?lat=51.83&lon=5.86&c='+str(random.randint(0,999999999999999)) ).text; dataRequest = dataRequest.replace('\r\n',' '); data = '{\"data\":\"'+dataRequest+'\"}';    pri(data);"
2023-12-12 18:29:30.907 WARNING (MainThread) [homeassistant.components.command_line] Empty reply found when expecting JSON data
2023-12-12 18:31:10.663 ERROR (MainThread) [homeassistant.components.automation.hassio_wash_notifications] *HASSIO - Wash notifications: Choose at step 2: choice 1: If at step 4: Error executing script. Service not found for call_service at pos 1: Service notify.mobile_app_user2_phone not found.
2023-12-12 18:31:10.664 ERROR (MainThread) [homeassistant.components.automation.hassio_wash_notifications] *HASSIO - Wash notifications: Choose at step 2: choice 1: Error executing script. Service not found for if at pos 4: Service notify.mobile_app_user2_phone not found.
2023-12-12 18:31:10.665 ERROR (MainThread) [homeassistant.components.automation.hassio_wash_notifications] *HASSIO - Wash notifications: Error executing script. Service not found for choose at pos 2: Service notify.mobile_app_user2_phone not found.
2023-12-12 18:33:30.894 ERROR (SyncWorker_8) [homeassistant.components.command_line.utils] Command failed (with return code 1): python3 -c "import requests; import json; import random; dataRequest = requests.get('https://gpsgadget.buienradar.nl/data/raintext?lat=51.83&lon=5.86&c='+str(random.randint(0,999999999999999)) ).text; dataRequest = dataRequest.replace('\r\n',' '); data = '{\"data\":\"'+dataRequest+'\"}';    pri(data);"
2023-12-12 18:33:30.895 WARNING (MainThread) [homeassistant.components.command_line] Empty reply found when expecting JSON data
2023-12-12 18:37:30.898 ERROR (SyncWorker_2) [homeassistant.components.command_line.utils] Command failed (with return code 1): python3 -c "import requests; import json; import random; dataRequest = requests.get('https://gpsgadget.buienradar.nl/data/raintext?lat=51.83&lon=5.86&c='+str(random.randint(0,999999999999999)) ).text; dataRequest = dataRequest.replace('\r\n',' '); data = '{\"data\":\"'+dataRequest+'\"}';    pri(data);"
2023-12-12 18:37:30.900 WARNING (MainThread) [homeassistant.components.command_line] Empty reply found when expecting JSON data
2023-12-12 18:39:56.929 WARNING (MainThread) [homeassistant.helpers.template] Template variable warning: 'title' is undefined when rendering '{{ title }}'
2023-12-12 18:39:57.254 WARNING (MainThread) [homeassistant.helpers.template] Template variable warning: 'title' is undefined when rendering '{{ title }}'
2023-12-12 18:39:57.257 ERROR (MainThread) [homeassistant.components.script.notify_people_who_are_present] Notify people who are home: If at step 2: Error executing script. Service not found for call_service at pos 1: Service notify.mobile_app_user2_phone not found.
2023-12-12 18:39:57.259 ERROR (MainThread) [homeassistant.components.script.notify_people_who_are_present] Notify people who are home: Error executing script. Service not found for if at pos 2: Service notify.mobile_app_user2_phone not found.
2023-12-12 18:39:57.263 ERROR (MainThread) [homeassistant.components.automation.all_rooms_rain_alert_when_window_open] All rooms - Rain alert when window open: If at step 4: Error executing script. Service not found for call_service at pos 1: Service notify.mobile_app_user2_phone not found.
2023-12-12 18:39:57.264 ERROR (MainThread) [homeassistant.components.automation.all_rooms_rain_alert_when_window_open] All rooms - Rain alert when window open: Error executing script. Service not found for if at pos 4: Service notify.mobile_app_user2_phone not found.
2023-12-12 18:41:30.930 ERROR (SyncWorker_4) [homeassistant.components.command_line.utils] Command failed (with return code 1): python3 -c "import requests; import json; import random; dataRequest = requests.get('https://gpsgadget.buienradar.nl/data/raintext?lat=51.83&lon=5.86&c='+str(random.randint(0,999999999999999)) ).text; dataRequest = dataRequest.replace('\r\n',' '); data = '{\"data\":\"'+dataRequest+'\"}';    pri(data);"
2023-12-12 18:41:30.932 WARNING (MainThread) [homeassistant.components.command_line] Empty reply found when expecting JSON data
2023-12-12 18:45:30.970 ERROR (SyncWorker_2) [homeassistant.components.command_line.utils] Command failed (with return code 1): python3 -c "import requests; import json; import random; dataRequest = requests.get('https://gpsgadget.buienradar.nl/data/raintext?lat=51.83&lon=5.86&c='+str(random.randint(0,999999999999999)) ).text; dataRequest = dataRequest.replace('\r\n',' '); data = '{\"data\":\"'+dataRequest+'\"}';    pri(data);"
2023-12-12 18:45:30.971 WARNING (MainThread) [homeassistant.components.command_line] Empty reply found when expecting JSON data
2023-12-12 18:49:30.917 ERROR (SyncWorker_1) [homeassistant.components.command_line.utils] Command failed (with return code 1): python3 -c "import requests; import json; import random; dataRequest = requests.get('https://gpsgadget.buienradar.nl/data/raintext?lat=51.83&lon=5.86&c='+str(random.randint(0,999999999999999)) ).text; dataRequest = dataRequest.replace('\r\n',' '); data = '{\"data\":\"'+dataRequest+'\"}';    pri(data);"
2023-12-12 18:49:30.920 WARNING (MainThread) [homeassistant.components.command_line] Empty reply found when expecting JSON data
2023-12-12 18:53:30.939 ERROR (SyncWorker_5) [homeassistant.components.command_line.utils] Command failed (with return code 1): python3 -c "import requests; import json; import random; dataRequest = requests.get('https://gpsgadget.buienradar.nl/data/raintext?lat=51.83&lon=5.86&c='+str(random.randint(0,999999999999999)) ).text; dataRequest = dataRequest.replace('\r\n',' '); data = '{\"data\":\"'+dataRequest+'\"}';    pri(data);"
2023-12-12 18:53:30.941 WARNING (MainThread) [homeassistant.components.command_line] Empty reply found when expecting JSON data
2023-12-12 18:57:31.064 ERROR (SyncWorker_8) [homeassistant.components.command_line.utils] Command failed (with return code 1): python3 -c "import requests; import json; import random; dataRequest = requests.get('https://gpsgadget.buienradar.nl/data/raintext?lat=51.83&lon=5.86&c='+str(random.randint(0,999999999999999)) ).text; dataRequest = dataRequest.replace('\r\n',' '); data = '{\"data\":\"'+dataRequest+'\"}';    pri(data);"
2023-12-12 18:57:31.068 WARNING (MainThread) [homeassistant.components.command_line] Empty reply found when expecting JSON data
2023-12-12 19:01:31.049 ERROR (SyncWorker_5) [homeassistant.components.command_line.utils] Command failed (with return code 1): python3 -c "import requests; import json; import random; dataRequest = requests.get('https://gpsgadget.buienradar.nl/data/raintext?lat=51.83&lon=5.86&c='+str(random.randint(0,999999999999999)) ).text; dataRequest = dataRequest.replace('\r\n',' '); data = '{\"data\":\"'+dataRequest+'\"}';    pri(data);"
2023-12-12 19:01:31.051 WARNING (MainThread) [homeassistant.components.command_line] Empty reply found when expecting JSON data
2023-12-12 19:05:31.058 ERROR (SyncWorker_7) [homeassistant.components.command_line.utils] Command failed (with return code 1): python3 -c "import requests; import json; import random; dataRequest = requests.get('https://gpsgadget.buienradar.nl/data/raintext?lat=51.83&lon=5.86&c='+str(random.randint(0,999999999999999)) ).text; dataRequest = dataRequest.replace('\r\n',' '); data = '{\"data\":\"'+dataRequest+'\"}';    pri(data);"
2023-12-12 19:05:31.059 WARNING (MainThread) [homeassistant.components.command_line] Empty reply found when expecting JSON data
2023-12-12 19:09:31.034 ERROR (SyncWorker_1) [homeassistant.components.command_line.utils] Command failed (with return code 1): python3 -c "import requests; import json; import random; dataRequest = requests.get('https://gpsgadget.buienradar.nl/data/raintext?lat=51.83&lon=5.86&c='+str(random.randint(0,999999999999999)) ).text; dataRequest = dataRequest.replace('\r\n',' '); data = '{\"data\":\"'+dataRequest+'\"}';    pri(data);"
2023-12-12 19:09:31.035 WARNING (MainThread) [homeassistant.components.command_line] Empty reply found when expecting JSON data
2023-12-12 19:13:31.000 ERROR (SyncWorker_4) [homeassistant.components.command_line.utils] Command failed (with return code 1): python3 -c "import requests; import json; import random; dataRequest = requests.get('https://gpsgadget.buienradar.nl/data/raintext?lat=51.83&lon=5.86&c='+str(random.randint(0,999999999999999)) ).text; dataRequest = dataRequest.replace('\r\n',' '); data = '{\"data\":\"'+dataRequest+'\"}';    pri(data);"
2023-12-12 19:13:31.002 WARNING (MainThread) [homeassistant.components.command_line] Empty reply found when expecting JSON data
2023-12-12 19:17:31.264 ERROR (SyncWorker_5) [homeassistant.components.command_line.utils] Command failed (with return code 1): python3 -c "import requests; import json; import random; dataRequest = requests.get('https://gpsgadget.buienradar.nl/data/raintext?lat=51.83&lon=5.86&c='+str(random.randint(0,999999999999999)) ).text; dataRequest = dataRequest.replace('\r\n',' '); data = '{\"data\":\"'+dataRequest+'\"}';    pri(data);"
2023-12-12 19:17:31.265 WARNING (MainThread) [homeassistant.components.command_line] Empty reply found when expecting JSON data
2023-12-12 19:21:31.212 ERROR (SyncWorker_1) [homeassistant.components.command_line.utils] Command failed (with return code 1): python3 -c "import requests; import json; import random; dataRequest = requests.get('https://gpsgadget.buienradar.nl/data/raintext?lat=51.83&lon=5.86&c='+str(random.randint(0,999999999999999)) ).text; dataRequest = dataRequest.replace('\r\n',' '); data = '{\"data\":\"'+dataRequest+'\"}';    pri(data);"
2023-12-12 19:21:31.214 WARNING (MainThread) [homeassistant.components.command_line] Empty reply found when expecting JSON data
2023-12-12 19:25:31.220 ERROR (SyncWorker_2) [homeassistant.components.command_line.utils] Command failed (with return code 1): python3 -c "import requests; import json; import random; dataRequest = requests.get('https://gpsgadget.buienradar.nl/data/raintext?lat=51.83&lon=5.86&c='+str(random.randint(0,999999999999999)) ).text; dataRequest = dataRequest.replace('\r\n',' '); data = '{\"data\":\"'+dataRequest+'\"}';    pri(data);"
2023-12-12 19:25:31.221 WARNING (MainThread) [homeassistant.components.command_line] Empty reply found when expecting JSON data
2023-12-12 19:29:31.214 ERROR (SyncWorker_9) [homeassistant.components.command_line.utils] Command failed (with return code 1): python3 -c "import requests; import json; import random; dataRequest = requests.get('https://gpsgadget.buienradar.nl/data/raintext?lat=51.83&lon=5.86&c='+str(random.randint(0,999999999999999)) ).text; dataRequest = dataRequest.replace('\r\n',' '); data = '{\"data\":\"'+dataRequest+'\"}';    pri(data);"
2023-12-12 19:29:31.215 WARNING (MainThread) [homeassistant.components.command_line] Empty reply found when expecting JSON data
2023-12-12 19:33:31.237 ERROR (SyncWorker_9) [homeassistant.components.command_line.utils] Command failed (with return code 1): python3 -c "import requests; import json; import random; dataRequest = requests.get('https://gpsgadget.buienradar.nl/data/raintext?lat=51.83&lon=5.86&c='+str(random.randint(0,999999999999999)) ).text; dataRequest = dataRequest.replace('\r\n',' '); data = '{\"data\":\"'+dataRequest+'\"}';    pri(data);"
2023-12-12 19:33:31.238 WARNING (MainThread) [homeassistant.components.command_line] Empty reply found when expecting JSON data
2023-12-12 19:37:31.491 ERROR (SyncWorker_7) [homeassistant.components.command_line.utils] Command failed (with return code 1): python3 -c "import requests; import json; import random; dataRequest = requests.get('https://gpsgadget.buienradar.nl/data/raintext?lat=51.83&lon=5.86&c='+str(random.randint(0,999999999999999)) ).text; dataRequest = dataRequest.replace('\r\n',' '); data = '{\"data\":\"'+dataRequest+'\"}';    pri(data);"
2023-12-12 19:37:31.493 WARNING (MainThread) [homeassistant.components.command_line] Empty reply found when expecting JSON data
2023-12-12 19:41:31.470 ERROR (SyncWorker_3) [homeassistant.components.command_line.utils] Command failed (with return code 1): python3 -c "import requests; import json; import random; dataRequest = requests.get('https://gpsgadget.buienradar.nl/data/raintext?lat=51.83&lon=5.86&c='+str(random.randint(0,999999999999999)) ).text; dataRequest = dataRequest.replace('\r\n',' '); data = '{\"data\":\"'+dataRequest+'\"}';    pri(data);"
2023-12-12 19:41:31.472 WARNING (MainThread) [homeassistant.components.command_line] Empty reply found when expecting JSON data
2023-12-12 19:45:31.470 ERROR (SyncWorker_4) [homeassistant.components.command_line.utils] Command failed (with return code 1): python3 -c "import requests; import json; import random; dataRequest = requests.get('https://gpsgadget.buienradar.nl/data/raintext?lat=51.83&lon=5.86&c='+str(random.randint(0,999999999999999)) ).text; dataRequest = dataRequest.replace('\r\n',' '); data = '{\"data\":\"'+dataRequest+'\"}';    pri(data);"
2023-12-12 19:45:31.471 WARNING (MainThread) [homeassistant.components.command_line] Empty reply found when expecting JSON data
2023-12-12 19:46:23.098 WARNING (MainThread) [homeassistant.helpers.script] Error in 'if[0]' evaluation: In 'numeric_state':
  In 'numeric_state' condition: unknown entity sensor.humidity_11
2023-12-12 19:49:20.417 ERROR (MainThread) [frontend.js.latest.202312082] Uncaught error from Chrome 119.0.0.0 on Windows 10
TypeError: Cannot read properties of undefined (reading 'createTimeRange')
/@videojs/http-streaming@2.16.0/dist/videojs-http-streaming.js:2541:52
/@videojs/http-streaming@2.16.0/dist/videojs-http-streaming.js:5:77
/@videojs/http-streaming@2.16.0/dist/videojs-http-streaming.js:6:3
2023-12-12 19:49:31.424 ERROR (SyncWorker_3) [homeassistant.components.command_line.utils] Command failed (with return code 1): python3 -c "import requests; import json; import random; dataRequest = requests.get('https://gpsgadget.buienradar.nl/data/raintext?lat=51.83&lon=5.86&c='+str(random.randint(0,999999999999999)) ).text; dataRequest = dataRequest.replace('\r\n',' '); data = '{\"data\":\"'+dataRequest+'\"}';    pri(data);"
2023-12-12 19:49:31.426 WARNING (MainThread) [homeassistant.components.command_line] Empty reply found when expecting JSON data
2023-12-12 19:50:43.035 ERROR (MainThread) [aiohttp.server] Error handling request
Traceback (most recent call last):
  File "/usr/local/lib/python3.11/site-packages/aiohttp/web_protocol.py", line 350, in data_received
    messages, upgraded, tail = self._request_parser.feed_data(data)
                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "aiohttp/_http_parser.pyx", line 557, in aiohttp._http_parser.HttpParser.feed_data
aiohttp.http_exceptions.BadHttpMessage: 400, message:
  Pause on PRI/Upgrade:

    b''
      ^
2023-12-12 19:53:31.467 ERROR (SyncWorker_4) [homeassistant.components.command_line.utils] Command failed (with return code 1): python3 -c "import requests; import json; import random; dataRequest = requests.get('https://gpsgadget.buienradar.nl/data/raintext?lat=51.83&lon=5.86&c='+str(random.randint(0,999999999999999)) ).text; dataRequest = dataRequest.replace('\r\n',' '); data = '{\"data\":\"'+dataRequest+'\"}';    pri(data);"
2023-12-12 19:53:31.468 WARNING (MainThread) [homeassistant.components.command_line] Empty reply found when expecting JSON data
2023-12-12 19:57:31.690 ERROR (SyncWorker_9) [homeassistant.components.command_line.utils] Command failed (with return code 1): python3 -c "import requests; import json; import random; dataRequest = requests.get('https://gpsgadget.buienradar.nl/data/raintext?lat=51.83&lon=5.86&c='+str(random.randint(0,999999999999999)) ).text; dataRequest = dataRequest.replace('\r\n',' '); data = '{\"data\":\"'+dataRequest+'\"}';    pri(data);"
2023-12-12 19:57:31.692 WARNING (MainThread) [homeassistant.components.command_line] Empty reply found when expecting JSON data
2023-12-12 20:00:00.100 ERROR (MainThread) [homeassistant.components.automation.hassio_medicine_reminder] *HASSIO - Medicine reminder: Choose at step 1: choice 3: Error executing script. Service not found for call_service at pos 1: Service notify.mobile_app_user2_phone not found.
2023-12-12 20:00:00.101 ERROR (MainThread) [homeassistant.components.automation.hassio_medicine_reminder] *HASSIO - Medicine reminder: Error executing script. Service not found for choose at pos 1: Service notify.mobile_app_user2_phone not found.
2023-12-12 20:01:31.662 ERROR (SyncWorker_5) [homeassistant.components.command_line.utils] Command failed (with return code 1): python3 -c "import requests; import json; import random; dataRequest = requests.get('https://gpsgadget.buienradar.nl/data/raintext?lat=51.83&lon=5.86&c='+str(random.randint(0,999999999999999)) ).text; dataRequest = dataRequest.replace('\r\n',' '); data = '{\"data\":\"'+dataRequest+'\"}';    pri(data);"
2023-12-12 20:01:31.664 WARNING (MainThread) [homeassistant.components.command_line] Empty reply found when expecting JSON data
2023-12-12 20:04:00.573 WARNING (MainThread) [homeassistant.helpers.template] Template variable warning: 'title' is undefined when rendering '{{ title }}'
2023-12-12 20:04:00.918 WARNING (MainThread) [homeassistant.helpers.template] Template variable warning: 'title' is undefined when rendering '{{ title }}'
2023-12-12 20:04:00.920 ERROR (MainThread) [homeassistant.components.script.notify_people_who_are_present] Notify people who are home: If at step 2: Error executing script. Service not found for call_service at pos 1: Service notify.mobile_app_user2_phone not found.
2023-12-12 20:04:00.921 ERROR (MainThread) [homeassistant.components.script.notify_people_who_are_present] Notify people who are home: Error executing script. Service not found for if at pos 2: Service notify.mobile_app_user2_phone not found.
2023-12-12 20:04:00.927 ERROR (MainThread) [homeassistant.components.automation.all_rooms_rain_alert_when_window_open] All rooms - Rain alert when window open: If at step 4: Error executing script. Service not found for call_service at pos 1: Service notify.mobile_app_user2_phone not found.
2023-12-12 20:04:00.928 ERROR (MainThread) [homeassistant.components.automation.all_rooms_rain_alert_when_window_open] All rooms - Rain alert when window open: Error executing script. Service not found for if at pos 4: Service notify.mobile_app_user2_phone not found.
2023-12-12 20:05:31.699 ERROR (SyncWorker_4) [homeassistant.components.command_line.utils] Command failed (with return code 1): python3 -c "import requests; import json; import random; dataRequest = requests.get('https://gpsgadget.buienradar.nl/data/raintext?lat=51.83&lon=5.86&c='+str(random.randint(0,999999999999999)) ).text; dataRequest = dataRequest.replace('\r\n',' '); data = '{\"data\":\"'+dataRequest+'\"}';    pri(data);"
2023-12-12 20:05:31.700 WARNING (MainThread) [homeassistant.components.command_line] Empty reply found when expecting JSON data
2023-12-12 20:07:13.906 WARNING (MainThread) [homeassistant.helpers.script] Error in 'if[0]' evaluation: In 'numeric_state':
  In 'numeric_state' condition: unknown entity sensor.humidity_11
2023-12-12 20:09:31.628 ERROR (SyncWorker_3) [homeassistant.components.command_line.utils] Command failed (with return code 1): python3 -c "import requests; import json; import random; dataRequest = requests.get('https://gpsgadget.buienradar.nl/data/raintext?lat=51.83&lon=5.86&c='+str(random.randint(0,999999999999999)) ).text; dataRequest = dataRequest.replace('\r\n',' '); data = '{\"data\":\"'+dataRequest+'\"}';    pri(data);"
2023-12-12 20:09:31.630 WARNING (MainThread) [homeassistant.components.command_line] Empty reply found when expecting JSON data
2023-12-12 20:13:31.664 ERROR (SyncWorker_7) [homeassistant.components.command_line.utils] Command failed (with return code 1): python3 -c "import requests; import json; import random; dataRequest = requests.get('https://gpsgadget.buienradar.nl/data/raintext?lat=51.83&lon=5.86&c='+str(random.randint(0,999999999999999)) ).text; dataRequest = dataRequest.replace('\r\n',' '); data = '{\"data\":\"'+dataRequest+'\"}';    pri(data);"
2023-12-12 20:13:31.665 WARNING (MainThread) [homeassistant.components.command_line] Empty reply found when expecting JSON data
2023-12-12 20:14:08.725 WARNING (MainThread) [homeassistant.helpers.template] Template variable warning: 'title' is undefined when rendering '{{ title }}'
2023-12-12 20:14:09.123 WARNING (MainThread) [homeassistant.helpers.template] Template variable warning: 'title' is undefined when rendering '{{ title }}'
2023-12-12 20:14:09.125 ERROR (MainThread) [homeassistant.components.script.notify_people_who_are_present] Notify people who are home: If at step 2: Error executing script. Service not found for call_service at pos 1: Service notify.mobile_app_user2_phone not found.
2023-12-12 20:14:09.126 ERROR (MainThread) [homeassistant.components.script.notify_people_who_are_present] Notify people who are home: Error executing script. Service not found for if at pos 2: Service notify.mobile_app_user2_phone not found.
2023-12-12 20:14:09.128 ERROR (MainThread) [homeassistant.components.automation.all_rooms_rain_alert_when_window_open] All rooms - Rain alert when window open: If at step 4: Error executing script. Service not found for call_service at pos 1: Service notify.mobile_app_user2_phone not found.
2023-12-12 20:14:09.129 ERROR (MainThread) [homeassistant.components.automation.all_rooms_rain_alert_when_window_open] All rooms - Rain alert when window open: Error executing script. Service not found for if at pos 4: Service notify.mobile_app_user2_phone not found.
2023-12-12 20:17:31.980 ERROR (SyncWorker_3) [homeassistant.components.command_line.utils] Command failed (with return code 1): python3 -c "import requests; import json; import random; dataRequest = requests.get('https://gpsgadget.buienradar.nl/data/raintext?lat=51.83&lon=5.86&c='+str(random.randint(0,999999999999999)) ).text; dataRequest = dataRequest.replace('\r\n',' '); data = '{\"data\":\"'+dataRequest+'\"}';    pri(data);"
2023-12-12 20:17:31.981 WARNING (MainThread) [homeassistant.components.command_line] Empty reply found when expecting JSON data
2023-12-12 20:21:31.921 ERROR (SyncWorker_7) [homeassistant.components.command_line.utils] Command failed (with return code 1): python3 -c "import requests; import json; import random; dataRequest = requests.get('https://gpsgadget.buienradar.nl/data/raintext?lat=51.83&lon=5.86&c='+str(random.randint(0,999999999999999)) ).text; dataRequest = dataRequest.replace('\r\n',' '); data = '{\"data\":\"'+dataRequest+'\"}';    pri(data);"
2023-12-12 20:21:31.921 WARNING (MainThread) [homeassistant.components.command_line] Empty reply found when expecting JSON data
2023-12-12 20:25:31.994 ERROR (SyncWorker_2) [homeassistant.components.command_line.utils] Command failed (with return code 1): python3 -c "import requests; import json; import random; dataRequest = requests.get('https://gpsgadget.buienradar.nl/data/raintext?lat=51.83&lon=5.86&c='+str(random.randint(0,999999999999999)) ).text; dataRequest = dataRequest.replace('\r\n',' '); data = '{\"data\":\"'+dataRequest+'\"}';    pri(data);"
2023-12-12 20:25:31.995 WARNING (MainThread) [homeassistant.components.command_line] Empty reply found when expecting JSON data
2023-12-12 20:29:32.162 ERROR (SyncWorker_8) [homeassistant.components.command_line.utils] Command failed (with return code 1): python3 -c "import requests; import json; import random; dataRequest = requests.get('https://gpsgadget.buienradar.nl/data/raintext?lat=51.83&lon=5.86&c='+str(random.randint(0,999999999999999)) ).text; dataRequest = dataRequest.replace('\r\n',' '); data = '{\"data\":\"'+dataRequest+'\"}';    pri(data);"
2023-12-12 20:29:32.164 WARNING (MainThread) [homeassistant.components.command_line] Empty reply found when expecting JSON data
2023-12-12 20:30:07.143 ERROR (MainThread) [homeassistant.components.template.template_entity] TemplateError('ValueError: Template error: float got invalid input 'unavailable' when rendering template '{{ (states.sensor.front_door_sensor_device_temperature.state | float - 3 ) | round(0) }}' but no default was specified') while processing template 'Template<template=({{ (states.sensor.front_door_sensor_device_temperature.state | float - 3 ) | round(0) }}) renders=40>' for attribute '_attr_native_value' in entity 'sensor.front_door_sensor_device_temperature_adj'
2023-12-12 20:30:07.154 ERROR (MainThread) [homeassistant.components.template.template_entity] TemplateError('ValueError: Template error: float got invalid input 'unavailable' when rendering template '{{ (states.sensor.desk_light_button_device_temperature.state | float - 8 ) | round(0) }}' but no default was specified') while processing template 'Template<template=({{ (states.sensor.desk_light_button_device_temperature.state | float - 8 ) | round(0) }}) renders=12>' for attribute '_attr_native_value' in entity 'sensor.desk_temp_adj'
2023-12-12 20:30:07.158 ERROR (MainThread) [homeassistant.components.template.template_entity] TemplateError('ValueError: Template error: float got invalid input 'unavailable' when rendering template '{{ (states.sensor.living_room_window_sensor_device_temperature.state | float - 4 ) | round(0) }}' but no default was specified') while processing template 'Template<template=({{ (states.sensor.living_room_window_sensor_device_temperature.state | float - 4 ) | round(0) }}) renders=20>' for attribute '_attr_native_value' in entity 'sensor.living_room_window_sensor_device_temperature_adj'
